---
title: "Population genetics"
author: |
  | Deependra Dhakal
  | Gokuleshwor Agriculture and Animal Science College
  | Tribhuwan University
  | \textit{ddhakal.rookie@gmail.com}
  | \url{https://rookie.rbind.io}
date: Academic year 2019-2020
output: 
  binb::monash:
    toc: true
    keep_tex: true
    slide_level: 2
    includes:
      in_header: beamer_header_template.tex
colortheme: monashwhite
outertheme: smoothtree
innertheme: rounded
innercolortheme: rose
outercolortheme: whale
linkcolor: red
urlcolor: lightgrayd
# titlefontsize: 22pt # breaks custom \lineskip
fontsize: 11pt
classoption: "aspectratio=169" # "compressed"
bibliography: [./../bibliographies.bib]
header-includes: 
  - \AtBeginSubsection{}
---

```{r setup, include=FALSE}
library(knitr)
require(tidyverse)
set.seed(453)
# invalidate cache when the package version changes
knitr::opts_chunk$set(tidy = FALSE, echo = FALSE, 
                  message = FALSE, warning = FALSE,
                  out.width = "45%", cache = TRUE, 
                  dev.args=list(bg=grey(0.9), pointsize=11))
options(knitr.table.format = "latex")
options(knitr.kable.NA = "", digits = 2)
options(kableExtra.latex.load_packages = FALSE)
theme_set(theme_bw())
```

# Gene and genotypes

## What is allele/gene ?

An allele/gene is the bit of DNA at the place called locus (the place on a chromosome where an allele resides). An allele is instantiation of a locus. But by orthology, a locus is not template for an allele. Similarly, a locus is not tangible, rather a map describing where to find a tangible thing, an allele on a chromosome. A diploid individual has two alleles at a particular autosomal locus.

<!-- ## Mechanisms by which alleles at same locus changes -->

<!-- 1. By origin: Same locus but different chromosome. -->

<!-- 2. By state: It is indicative of the context they are put in. i.e. DNA sequence or amino acid sequence. Same amino acid sequence in some alleles may arise due to different DNA sequences (Redundancy of genetic code). -->

<!-- 3. By descent: In practice, we are often concerned with relatively short time in past and are content to say that two alleles differ by descent if they do not share common ancestor in say, the past 10 generations. Two alleles different by descent may or may not be different by state because of mutation. -->

<!-- ## -->

<!-- Converse of the mechanisms which cause differences in alleles are termed as identical by origin, stage or descent. Diploid individuals are said to be heterozygous at a locus if two alleles at that locus are different by state. If we are studying proteins, we may call an individual homozygous at a locus when the protein sequence of the two alleles are identical, even if their DNA sequences differ. -->

<!-- ```{r, message=FALSE, warning=FALSE, fig.cap="An example of pedigree tree showing single pedigree family with 14 subjects", fig.width=8, fig.height=5, echo=FALSE} -->
<!-- require(kinship2) -->

<!-- # load data -->
<!-- data("sample.ped") -->
<!-- # define pedigree list object -->
<!-- sam_ped <- pedigree(id = sample.ped$id, dadid = sample.ped$father, momid = sample.ped$mother, sex = sample.ped$sex, famid = sample.ped$ped) -->

<!-- # pedigree plot -->
<!-- sam_ped[2] %>%  -->
<!--   plot(cex = 0.8) -->
<!-- ``` -->


<!-- ## Kreitman's sample -->

<!-- Kreitman's sample contain II alleles that differ by origin. How many alleles differ by state? If we were interested in the full DNA sequence, the sample contains six alleles that differ by state. If we were interested in the proteins, then sample contains only two alleles that differ by state. Of the two protein alleles, the one with a lysine at position 192 makes up $\frac{6}{10} = 0.55$ of the alleles. The usual way to say this is that the allele frequency is an estimate of the population allele frequency. It's not a particularly precise estimate because of the small sample size. A rough approximation to the 95% confidence interval for a proportion is, -->

<!-- $$ -->
<!-- \hat{p} \pm 1.96 \sqrt{\frac{\hat{p}(1-\hat{p})}{n}} -->
<!-- $$ -->

<!-- Where $\hat{p}$ is the estimate of proportion, 0.55 in our case and $n$ is the sample size. Thus, the probability that the populatoin allele frequency falls within the interval (0.26, 0.84) is 0.95. If more precise estimate is needed, the sample size would have to be increased. -->

## Genotype and allele frequencies

In A loci, suppose, two alleles $A_1$ and $A_2$ are present in a diploid organism the genotype and genotypic frequency of segregating population will be;

$$
\begin{aligned}
\textrm{Genotype} \hspace{20pt} & A_1A_1 & A_1A_2 \hspace{20pt} & A_2A_2 \\
\textrm{Relative frequency} \hspace{20pt} & x_{11} & x_{12} \hspace{20pt} & x_{22}
\end{aligned}
$$

As, relative frequencies must add to 1,

$$
x_{11} + x_{12} + x_{22} = 1
$$

The order of subscripting heterozygous is arbitrary.

Frequency of $A_1$ allele in the population is,

$$
p = x_{11} + \frac{1}{2}x_{12}
$$

##

and frequency of $A_2$ allele is,

$$
q = 1-p = x_{22} + \frac{1}{2}x_{12}
$$

Measure of each allele frequency can be thought of as independent events. For e.g., for allele $p$ to be selected;

$$
p = \left(x_{11} \times \frac{1}{P(p_{A_1A_1})}\right) + \left(x_{12} \times \frac{1}{2}\right) + (x_{22} \times 0)
$$

Where, $P(p, A_1A_1)$ is the probability of getting $p$ allele from $A_1A_1$ genotype, for loci with more than two alleles, frequency of $i^{th}$ allele will be called $p_i$. Frequency of $A_iA_j$ genotype will be called $x_ij$ for heterozygotes, $i\neq j$ and, by convention, $i<j$. 

##

If there are $n$ alleles,

$$
\begin{aligned}
1 &= x_{11} + x_{22} + x_{33} + ... + x_{nn} + x_{12} + x_{13} + x_{(n-1)n} \\
  &= \sum^n_{i=1}\sum^n_{j\geq i}{x_{ij}}
\end{aligned}
$$

The frequency of $i^th$ allele is

$$
p_i = x_{ii} + \frac{1}{2}\sum^{i-1}_{j = 1}{x_{ji}} + \frac{1}{2}\sum^n_{j = i+1}{x_{ij}}
$$

# Hardy-Weinberg law

##

- In a random mating population (in which each male gamete has an equal chance of mating with any female gamete), when taken a loci with differences in alleles (A/a), following genotypes are possible: $AA$, $Aa$ and $aa$. 
- With the corresponding frequencies of $p^2$, $2pq$ and $q^2$, respectively for each of the genotypes, and bearing that gene frequencies must add up to unity, $p^2+2pq+q^2 = 1$. This mathematical relationship is called the **Hardy-Weinberg equilibrium**.
- Two scientists showed that the frequency of genotypes in a population depends on the frequency of genes in the preceding generation, not on the frequency of the genotypes.

##

- In each subsequent generation following thereafter, however, gene and genotypic frequencies will remain unchanged, provided that:
  1. Random mating occurs in a very large diploid population;
  2. Allele A and allele a are equally fit (one does not confer a superior trait than the other);
  3. There is no differential migration of one allele into or out of the population;
  4. The mutation rate of allele A is equal to that of allele a.

## Conservation of gene frequencies

\small

- Let us presume that among humans, the difference between those who can and those who cannot taste the chemical phenyl-thiocarbamate (PTC) resides in a single gene difference with two alleles, $T$ and $t$.
- The allele for tasting, $T$, is dominant over $t$, so that heterozygotes, $Tt$ are tasters, and the only nontasters are $tt$.
- If we were to choose an initial population composed of an arbitary number of each genotype, we may ask what will be the frequency of these genes after many generations.
- Let us, for example, place upon an island a group of children in the ratio $.40TT:.40Tt:.20tt$. The gene frequencies in this newly formed population are therefore, $.4 + .2 = .6 T$, and $.2 + .2 = .4t$.
- Let us also assume that the number of individuals in the population is large, and the tasting or nontasting has no effect upon survival (viability), fertility, or attraction between the sexes.

##

- As these children mature, they will choose their mates at random from those of the opposite sex regardless of their tasting abilities.
- Matings between any two genotypes can then be predicted solely on the basis of the frequency of those genotypes in the population.
- Table \ref{tab:random-mating-monogene-gene-freq} shows matings in all possible combination.

```{r random-mating-monogene-gene-freq}
tribble(
  ~"TT", ~"Tt", ~"tt", 
  .16, .16, .08, 
  .16, .16, .08, 
  .08, .08, .04
) %>% 
  add_column(.before = 1, gametes = colnames(.)) %>% 
  kable(booktabs = TRUE, escape = TRUE, caption = "Types of random-mating combinations and their relative frequencies in a population containing .40TT, .40Tt, and .20tt genotypes") %>% 
  kableExtra::kable_styling(position = "center")
```

##

```{r}
tribble(~"Type of mating", ~"Frequency of mating", 
        "$TT \\times TT$", .16,
        "$TT \\times Tt$", .32,
        "$TT \\times tt$", .16,
        "$Tt \\times Tt$", .16,
        "$Tt \\times tt$", .16,
        "$tt \\times tt$", .04) %>% 
  add_column(.after = 2, TT = c("all(.16)", "1/2(.32)", NA, "1/4(.16)", NA, NA)) %>% 
  add_column(.after = 3, Tt = c(NA, "+1/2(.32)", "all (.16)", "+1/2 (.16)", "1/2(.16)", NA)) %>% 
  add_column(.after = 4, tt = c(NA, NA, NA, "+1/4(.16)", "+1/2(.16)", "all (.04)")) %>% 
  kable(booktabs = TRUE, escape = FALSE, caption = "Relative frequencies of the different kinds of offspring produced by the matings") %>% 
  kableExtra::kable_styling(position = "center", latex_options = "striped") %>% 
  kableExtra::add_header_above(header = c("Parents" = 2, "Offspring ratio" = 3))
```

##

- Note that although the frequencies of genotypes have been altered by random mating, the gene frequencies have not changed.
- For the $T$ gene frequency is equal to $.36 + 1/2(.48) = .60$, and the frequency of $t$ is $.16 + 1/2(.48) = .40$. 
- No matter what the initial frequencies of the three genotypes, the gene frequencies of the next generation will be the same as those of parental generation.

## Assertion

1. Under conditions of random mating ( _panmixis_) in a large population where all genotypes are equally viable, gene frequencies of a particular generation depend upon the gene frequencies of the previous generation and not upon the _genotype frequencies_.
2. The frequencies of different genotypes produced through random mating depend only upon the gene frequencies.
- After the first generation of random mating, genotype frequencies also remain stable. i.e., equilibrium.

## Formal proof

```{r}
tribble(~"Type of mating", ~"Frequency of mating", 
        "$TT \\times TT$", "$p^2 \\times p^2 = p^4$",
        "$TT \\times Tt$", "$2 \\times p^2 \\times 2pq = 4p^3q$",
        "$TT \\times tt$", "$2 \\times p^2 \\times q^2 = 2p^2q^2$",
        "$Tt \\times Tt$", "$2pq \\times 2pq = 4p^2q^2$",
        "$Tt \\times tt$", "$2 \\times 2pq \\times q^2 = 4pq^3$",
        "$tt \\times tt$", "$q^2 \\times q^2 = q^4$") %>% 
  add_column(.after = 2, TT = c("$p^4$", "$2p^3q$", NA, "$p^2q^2$", NA, NA)) %>% 
  add_column(.after = 3, Tt = c(NA, "$2p^3q$", "$2p^2q^2$", "$2p^2q^2$", "$2pq^3$", NA)) %>% 
  add_column(.after = 4, tt = c(NA, NA, NA, "$p^2q^2$", "$2pq^3$", "$q^4$")) %>% 
  add_row("Type of mating" = NA,
          "Frequency of mating" = "$p^2(p^2 + 2pq + q^2) + 2pq(p^2 + 2pq + q^2) + q^2(p^2 + 2pq + q^2) = p^2 + 2pq + q^2 = (p + q)^2 = 1$",
          "TT" = "$p^4 + 2p^3q + p^2q^2 = p^2(p^2 + 2pq + q^2) = p^2$",
          "Tt" = "$2p^3q + 4p^2q^2 + 2pq^3 = 2pq(p^2 + 2pq + q^2) = 2pq$",
          "tt" = "$p^2q^2 + 2pq^3 + q^4 = q^2(p^2 + 2pq + q^2) = q^2$") %>%
  # mutate_at(c(2,3,4,5), ~replace(., 6, kableExtra::cell_spec(., underline = TRUE, format = "latex", escape = FALSE))) %>% # this has some troubles, use row spec instead
  kable(booktabs = TRUE, escape = FALSE, caption = "Mating combinations and frequencies of offspring produced under conditions of random mating when genotypic frequencies are $p^2TT$, $2pqTt$, and $q^2tt$") %>% 
  kableExtra::kable_styling(position = "center", latex_options = "striped", font_size = 6) %>% 
  kableExtra::column_spec(1:5, width = c("6em", "12em", "8em", "8em", "8em")) %>% 
  kableExtra::row_spec(6, hline_after = TRUE) %>% 
  kableExtra::add_header_above(header = c("Parents" = 2, "Offspring ratio" = 3))
```

## Problem

1. Three jokers went to the jungle, but they couldn't see any tree. How ? (Context: Development in Nepal!)

<!-- 1. How many different genotypes are there at a locus with n alleles that differ by state? -->

<!-- 2. Derive the hardy weinberg law for a sex-linked locus. Let the initial frequency A, in female be $p_f$ and in males be $p_m$. Follow the two allele frequencies in successive generations untill you understand the allele frequency dynamics. Then, jump ahead and find the equilibrium genotype frequencies in females and males. Finally, graph the male and female allele frequencies over several generations for a population that is started with all $A_1A_1$ females ($p_f = 1$) and A_2 males ($p_m = 0$) -->

3. A population is consisted of 200 plants. Out of them, 100 plants are of Aa, 50 plants are of AA and 50 plants are of aa genotypes. This is a random mating population and in this population the frequencies of these three genotypes are at H-W equilibrium state. After $5^{th}$ generations of random mating, plants having genotypes AA, Aa and aa are found in 500, 300 and 200 numbers respectively. Are they still in H-W equilibrium? Test the result with the help of $\chi^2$ goodness of fit test.

<!-- ## Solution -->

<!-- 1. When there are n alleles, there are n homozygous genotypes, $A_iA_i$, $i = 1, 2, ...., n$. If we first view an $A_iA_j$ heterozygote as distinct from an $A_jA_i$ heterozygote, there are $n(n-1)$ such heterozygotes. The actual number of heterozygotes will be one half this number, or $\frac{n(n-1)}{2}$. Thus, the total number of genotypes is $\frac{n+n(n-1)}{2}= \frac{n(n+1)}{2}$. -->

<!-- 2. As males get their X-chromosomes from their mother, the frequency of A_1 in males is always equal to the frequency in females in the previous generation. As a female gets one X from her mother and one from her father, the allele freq in females is always the average of the male and female frequencies in the previous generation. Thus, the allele frequencies over the first three generation are as follows. -->

<!-- ## -->

<!-- ```{r three-gen-freq, echo=FALSE} -->
<!-- require(tidyverse) -->
<!-- three_gen_freq <- tribble(~"Generation" , ~"Females" , ~"Males" , ~"Female-male", -->
<!-- 1 , "$p_f$" , "$p_m$" , "$p_f - p_m$", -->
<!-- 2 , "$\\frac{p_f + p_m}{2}$" , "$p_m$" , "$-\\frac{p_f -p_m}{2}$", -->
<!-- 3 , "$\\frac{p_f + \\frac{p_f + p_m}{2}}{2}$" , "$\\frac{p_f+ p_m}{2}$" , "$\\frac{p_f -p_m}{2}$" -->
<!-- ) -->

<!-- three_gen_freq %>%  -->
<!--   knitr::kable(booktabs = TRUE, escape = FALSE) %>%  -->
<!--   kableExtra::kable_styling(font_size = 6) -->
<!-- ``` -->

<!-- Two important things emerge from the table. First, the overall allele frequecy, -->

<!-- $$ -->
<!-- p = \frac{2}{3}p_f + \frac{1}{3}p_m -->
<!-- $$ -->

<!-- does not change over time. (Convince yourself that this is so by calculating p in generations 2 and 3). Second, the difference between the allele frequencies in females and males is halved each generation, as recorded in table. Taken together, these two observations show that eventually the allele frequencies in male and females will converge to $p$. At that time, the genotype frequencies in females will be at Hardy-Weinberg frequencies. -->

## Solution

3. Here, the population of 200 plants is stated to be in H-W equilibrium; we already have equilibrium frequencies. Hence a $\chi^2$ test for would show whether or not both the populations are same or have diverged from H-W equilibrium state (i.e. observed frequncy of population after 5th generation is same or different than expected population frequency at initial condition). For facilitating comparison, we convert the given frequencies of observed genotypes (that of $5^{th}$ generation) to the add upto current population count (200 individual). 

##

Thus observed frequencies are AA: `r 500/1000*200`; Aa: `r 300/1000*200` and aa: `r 200/1000*200`.

Note, however, we commonly compute the expected frequency based on the expected ratios. Therefore it also imperative to show the expected frequency as the proportion of total count of observed frequency.

Now we construct contingency table, as shown in Table \ref{tab:hw-equilibrium-independence-chi}.

```{r hw-equilibrium-independence-chi, echo=FALSE}
hw_contingency_chi <- tribble(~"Dominant (AA)", ~"Homozygous dominant (Aa)", ~"Recessive (aa)",
                           100, 50, 50,
                           100, 60, 40)

hw_contingency_chi %>% 
  add_column(.before = 1, "  " = c("$1^{st}$", "$5^{th}$")) %>% 
  add_column(.before = 1, " Gen " = "Generation") %>% 
  kable(format = "latex", booktabs = TRUE, caption = "2x3 contingency table of frequency of genotypes at equilibrium generation and at 5th generation of mating", 
        escape = FALSE, longtable = TRUE, 
        col.names = c(" ", " ", "Dominant (AA)", "Homozygous dominant (Aa)", "Recessive (aa)")) %>% 
  kableExtra::kable_styling(latex_options = "hold_position") %>% 
  kableExtra::add_header_above(c("  " = 2, "Genotype frequency" = 3)) %>% 
  kableExtra::collapse_rows(columns = 1)

hw_chi_test <- chisq.test(x = hw_contingency_chi, correct = FALSE)
```

##

Here since the number of df is `r hw_chi_test$parameter`, we do not apply the Yate's correction. After computation, we find $\chi^2$ = `r hw_chi_test$statistic` with probability of `r hw_chi_test$p.value` which is well within the confidence band of 0.95 to 0.05. We fail to reject the null hypothesis that two observations were taken from same populations. Thus, we conclude that even after $5^{th}$ generation of mating the population continues to be in HW equilibrium state.

<!-- # Conditions that affect equilibrium -->

<!-- ## Population size -->

<!-- The Hardy-Weinberg equilibrium requires a large random mating population (among other factors as previously indicated) to be true. However, in practice, the law has been found to be approximately true for most of the genes in most cross-pollinated species, except when non-random mating (e.g., inbreeding and assortative mating) occur. Whereas inbreeding is a natural feature of self-pollinated species, assortative mating can occur when cross-pollinated species are closely spaced in the field. -->

<!-- ## Multiple loci -->

<!-- Research has shown that it is possible for alleles at two loci to be in random mating frequencies and yet not in equilibrium with respect to each other. Furthermore, equilibrium between two loci is not attained after one generation of random mating as the Hardy-Weinberg law concluded but is attained slowly over many generations. Also, the presence of genetic linkage will further slow down the rate of attainment of equilibrium (Figure ??). If there is no linkage ($c = 0.5$), the differential between actual frequency and the equilibrium frequency is reduced by 50% in each generation. At this rate, it would take about seven generations to reach approximate equilibrium. However, at $c = 0.01$ 01, and $c = 0.001$, it would take about 69 and 693 generations, respectively, to reach equilibrium. A composite gene frequency can be calculated for genes at the two loci. For example, if the frequency at locus $Aa = 0.2$ and that for locus $bb = 0.7$, the composite frequency of a genotype $Aabb = 0.2 \times 0.7 = 0.14$. -->

# Factors affecting gene frequency

##

- Two major types of process identified:
  1. Systematic: Predictable in both direction and in amount
  2. Dispersive: Predictable only in amount

## Migration

Migration is important in small populations. It entails the entry of individuals into an existing population from outside. Because plants are sedentary, migration, when it occurs naturally, is via pollen transfer (gamete migration). The impact that this immigration will have on the recipient population will depend on the immigration rate and the difference in gene frequency between the immigrants and natives. Mathematically, $\Delta q = m(q_m - q_o)$, where $\Delta q$ = the change in frequency of genes in the new mixed population, $m$ = number of immigrants, $q_m$ = the gene frequency of the immigrants, and $q_o$ = the gene frequency of the bost. Plant breeders employ this process to change frequencies when they undertake introgression of genes into their breeding populations. The breeding implication is that for open-pollinated (outbreeding) species, the frequency of the immigrant gene may be low, but its effect on the host gene and genotypes could be significant.

## Mutation

Natural mutations are generally rare. A unique mutation (non-recurrent mutation) would have little impact on gene frequencies. Mutations are generally recessive in gene action, but the dominant condition may also be observed. Recurrent mutation (occurs repeatedly at a constant frequency) may impact gene frequency of the population. Natural mutations are of little importance to practical plant breeding. However, breeders may artificially induce mutation to generate new variability for plant breeding.

## Selection

- Selection is the most important process by which plant breeders alter population gene frequencies. Its effect is to change the mean value of the progeny population from that of the parental population. This change may be greater or lesser than the population mean, depending on the trait of interest. For example, breeders aim for higher yield but may accept and select for less of a chemical factor in the plant that may be toxic in addition to the high yield. For selection to succeed:
  1. there must be phenotypic variation for the trait to allow differences between genotypes to be observed;
  2. the phenotypic variation must at least be partly genetic.

## Random genetic drift

\small

- Nondirectional forces that arises from variable sampling of the gene pool each generation is known as random genetic drift.
- It is caused by the fact that real population are limited in size rather than infinite, so that gene-frequency changes occur because of sampling errors.
- For instance, if only a few parents are chosen to begin a new generation, such a small sample of genes may deviate widely from the gene frequency of the previous generation.
- The extent of the deviation in both cases can be measured mathematically by the standard deviation of a proportion ($\sigma = \sqrt{\frac{pq}{N}}$). Here $p$ is the frequency of one allele, $q$ of the other, and $N$ the number of genes sampled.
- For diploid parents, each carrying two genes, $\sigma = \sqrt{\frac{pq}{2N}}$, where $N$ is the number of actual parents.
- For example, if we begin with a large diploid population, where $p = q = .5$, and continue this population each generation by using 5000 parents, then $\sigma = \sqrt{(.5)(.5)/10000} = \sqrt{.000025} = 0.05$. The values of such populations will therefore fluctuate mostly (68% of the time), around $.5 \pm .005$, or between $0.495$ and $0.505$. On the other hand, a choice of only two parents as "founders" will produce a standard deviation of $\sqrt{(.5)(.5)/4} = 0.25$ or values of $.5 \pm .25$ (.25 to .75).

## Effect of mating system on selection

1. Random mating
2. Non random mating:
  - Genetic assortative mating: mating occurs such that the mating pair has the same phenotype more often than would occur by chance.
  - Phenotypic assortative mating: the breeder selects and mates individuals on the basis of their phenotypic resemblance to each other compared to the rest of the population.
  - Disassortative mating: may be genetic or phenotypic; entails mating individuals that are less closely related than they would under random mating (genotypic) or breeder may select individuals with contrasting phenotypes for mating (phenotypic).

# Heterosis and Inbreeding depression

## Heterosis (Hybrid vigor)

- Hybrid vigor may be defined as the increase in size, vigor, fertility, and overall productivity of a hybrid plant over the mid-parent value (average performance of the two parents). 
- It is calculated as the difference between the crossbred and inbred means:

$$\text{Hybrid vigour} = \frac{F_1-\frac{(P_1+P_2)}{2}}{\frac{(P_1+P_2)}{2}}$$

- The estimate is usually calculated as a percentage.
- The synonymous term, heterosis, was coined by G.H. Shull.
- Advantageous hybrid vigor is observed more frequently when breeders cross parents that are genetically diverse; When two inbred lines of outbred species are crossed.
- The practical definition of heterosis is hybrid vigor that greatly exceeds the better or higher parent in a cross.
- Hybrid breeding in maize quadrupled yields of maize in US between 1930s and 1970s.

## Inbreeding depression

- Inbreeding depression is reduction in fitness as a direct result of inbreeding. 
- In theory, the heterosis observed on crossing is expected to be equal to the depression upon inbreeding, considering a large number of crosses between lines derived from a single base population. 
- In practice, plant breeders are interested in heterosis expressed by specific crosses between selected parents, or between populations that have no known common origin.
- Reduction in fitness is usually manifested as a reduction in vigor, fertility, and productivity. 
- The effect is more severe in the early generations (5-8). 
- Plants including onions, sunflower, cucurbits, and rye are more tolerant of inbreeding with minimal consequences of inbreeding depression. 
- Plants such as alfalfa and carrot are highly intolerant of inbreeding.

##

- Inbreeding is measured by the coefficient of inbreeding (F), which is the probability of identity of alleles by descent. The range of F is zero (no inbreeding; random mating) to one (prolonged selfing).
- An unfit (deleterious) recessive allele is fairly quickly reduced in frequency but declines slowly thereafter. 
- On the other hand, an unfit dominant allele is rapidly eliminated from the population, while an intermediate allele is reduced more rapidly than a recessive allele because the former is open to selection in the heterozygote.
- The consequence of these outcomes is that unfit dominant or intermediate alleles are rare in cross-breeding populations, while unfit recessive alleles persist because they are protected by their recessiveness.

##

- A measure of Inbreeding Depression is obtained through generational mean analysis. It is calculated as:

$$
\text{Inbreeding depression (ID)} = \frac{F_1 - F_2}{F_1} \times 100\%
$$

##

- In Figure \ref{fig:inbreeding-coefficient} (a) there is no inbreeding because there is no common ancestral pathway to the individual, A (i.e., all parents are different).
- However, in Figure \ref{fig:inbreeding-coefficient} (b) inbreeding exists because B and C have common parents (D and E), that is, they are full sibs.
- To calculate the amount of inbreeding, the standard pedigree is converted to an arrow diagram, as shown in \ref{fig:inbreeding-coefficient} (c). 
- Each individual contributes 1/2 of its genotype to its offspring. The *coefficient of relationship* (R) is calculated by summing up all the pathways between two individuals through a common ancestor as: $R_{BC} = \sum{\left(\frac{1}{2}\right)^s}$ , where s is the number of steps (arrows) from B to the common ancestor and back to C. For example, B and C probably inherited $(1/2)(1/2) = 1/4$ of their genes in common through ancestor D. Similarly, B and C probably inherited 1/4 of their genes in common through ancestor E. 
- The coefficient of relationship between B and C, as a result of common ancestry, is hence $R_{BC} = 1/4 + 1/4 = 1/2 = 50\%$

##

```{r inbreeding-coefficient, fig.cap="Pedigree diagrams can be drawn in the standard form (a, b) or converted to into an arrow diagram (c).", fig.align="center", out.width="45%"}
# pdftools::pdf_convert("./../../references_plb/Principles of Plant Genetics and Breeding.pdf", pages = 64, format = "png", filenames = "./images/arrow_diagram.png", dpi = 300)
knitr::include_graphics("../images/arrow_diagram.png")
```

## Genetic basis of heterosis

- To explain the genetic basis for why fitness lost on inbreeding tends to be restored upon crossing, two theories have been proposed.
  - Dominance theory: C.G. Davenport in 1908 and later by I.M. Lerner, 
  - Overdominance theory: Shull in 1908 and later by K. Mather and J.L. Jinks. 
- A third theory, the mechanism of epistasis (non-allelic gene interactions), has also been proposed.

<!-- ## Dominance theory -->

<!-- - The dominance theory assumes that vigor in plants is conditioned by dominant alleles, recessive alleles being deleterious or neutral in effect.  -->
<!-- - It follows: a genotype with more dominant alleles will be more vigorous than one with few dominant alleles. -->
<!-- - Consequently, crossing two parents with complementary dominant alleles will concentrate more favorable alleles in the hybrid than either parent.  -->
<!-- - In practice, if too many deleterious alleles are present, it makes it difficult to inbreed to recover sufficient loci with homozygous dominant alleles.  -->
<!-- - Inbreeding depression occurs upon selfing because the deleterious recessive alleles that are protected in the heterozygous condition (heterozygous advantage) become homozygous and are expressed.  -->
<!-- - Assume that each dominant genotype contributes two units to the phenotype, while a recessive genotype contributes one unit. -->
<!-- - A cross between two inbred parents could produce the following outcome: -->

<!-- ## -->

<!-- ```{r dominance-theory, fig.align="center", out.width="auto", out.height="200px"} -->
<!-- # pdftools::pdf_convert("./../../references_plb/Principles of Plant Genetics and Breeding.pdf", pages = 368, format = "png", filenames = "./images/dominance_theory.png", dpi = 300) -->
<!-- knitr::include_graphics("./images/dominance_theory.png") -->
<!-- ``` -->

<!-- ## Overdominance theory -->

<!-- - The phenomenon of the heterozygote being superior to the homozygote is called overdominance (i.e., heterozygosity _per se_ is assumed to be responsible for heterosis).  -->
<!-- - Theory assumes that the alleles of a gene (e.g., A, a) are contrasting but each has a different favorable effect in the plant.  -->
<!-- - A genotype with more heterozygous loci would be more vigorous than one with less heterozygotes. -->
<!-- - Consider a quantitative trait conditioned by four loci, and assume that recessive, heterozygote, and homozygote dominants contribute 1, 2, and 1.5 units to the phenotypic value, respectively: -->

<!-- ```{r overdominance-theory, fig.align="center", out.width="auto", out.height="200px"} -->
<!-- # pdftools::pdf_convert("./../../references_plb/Principles of Plant Genetics and Breeding.pdf", pages = 369, format = "png", filenames = "./images/overdominance_theory.png", dpi = 300) -->
<!-- knitr::include_graphics("./images/overdominance_theory.png") -->
<!-- ``` -->

## Biometrics of heterosis

1. Better parent heterosis (Heterobeltiosis)
$$Hybrid~vigour = \frac{F_1-Better~parent}{Better~parent}$$
2. Mid parent heterosis
$$Hybrid~vigour = \frac{F_1-\frac{(P_1+P_2)}{2}}{\frac{(P_1+P_2)}{2}}$$
3. Commercial heterosis
$$Hybrid~vigour = \frac{F_1-Commercial~Hybrid}{Commercial~Hybrid}$$

## Types of hybrids

- Commercial applications of hybrid breeding started with a cross of two inbred lines (a single cross - AxB) and later shifted to the more economic double cross, ([AxB]x[CxD]) and then back to a single cross. 
- Other parent combinations in hybrid development have been proposed, including the three-way cross ([AxB]xC) and modified versions of the single cross, in which closely related crosses showed that the single cross was superior in performance to the other two in terms of average yield. 
- However, it was noted also that the genotype x environment interaction (hybrid x environment) variability was more than twice that for the double crosses, while the mean variability for the three-way cross being intermediate.

##

- This indicated that the single crosses were more sensitive or responsive to environmental conditions than the other crosses. 
- Whereas high average yield is important to the producer, consistency in performance across years and locations (i.e., yield stability) is also important. 
- Double and three-way crosses have a more genetically divergent population for achieving buffering. 
- Today commercial hybrids are predominantly single cross, of best combining parental inbred lines.
- For outline of mating scheme, See Lecture 7 on "Hybridization techniques and its consequences" (Course: Introductory plant breeding, $4^{th}$ semester, BScAg). 

# Numerical problems

## Problem

In blackgram, grain yield of parents ($P_1$ and $P_2$) their $F_1$ and $F_2$ progenies are given below:

```{r}
tribble(
  ~"Parent 1", ~"Parent 2", ~"F1 hybrid", ~"F2 progeny",
  18.94, 22.69, 29.38, 15.18
) %>% 
  kable(booktabs = TRUE) %>% 
  kableExtra::kable_styling(font_size = 6, position = "center")
```

Calculate average heterosis, heterobeltiosis and inbreeding depression.

## Solution

\begin{equation}
\begin{aligned}
\text{Mid parent heterosis} &= \frac{F_1 - MP}{MP} \times 100\% \\
\text{Here, Value of } F_1 &= 29.38 \\
\text{Mean of parents (MP)} &= \frac{18.9+22.69}{2} = 20.81 \\
\text{Mid parent heterosis } &= \frac{29.38-20.81}{20.81} \times 100\% = 41.12\% \\
\text{Heterobeltiosis} &= \frac{F_1-BP}{BP} \times 100\% = \frac{29.38-22.69}{22.69} \times 100\% = 29.48\% \\
\text{Inbreeding depression} &= \frac{F_1-F_2}{F_1} \times 100\% = \frac{29.38-15.18}{15.18} \times 100\%= 48.33\%
\end{aligned}
\nonumber
\end{equation}

# Bibliography

## References
