---
title: "Numerical Problems"
author: |
  | Deependra Dhakal
  | Gokuleshwor Agriculture and Animal Science College, Baitadi, Nepal
date: "9/18/2020"
output: 
  bookdown::pdf_document2:
    keep_tex: yes
    toc: false
    includes:
      in_header: gen_solutions_header.tex
documentclass: exam
classoption: "answers,addpoints,12pt"
geometry: "top=1.5cm,bottom=1.5cm,left=1.5cm,right=1.5cm"

---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, include = TRUE, message = FALSE, warning = FALSE)
options(knitr.kable.NA = '')
require(tidyverse)
```

## Problem 4: Effective crossing

- A corn geneticist has three pure lines of genotypes a/a; B/B; C/C, A/A; b/b; C/C, and A/A; B/B; c/c. All the phenotypes determined by a, b and c will increase the market value of the corn; so, naturally, he wants to combine them all in one pure line of genotype a/a; b/b; c/c.
  1. Outline an effective crossing program that can be used to obtain the a/a; b/b; c/c pure line.
  2. At each stage, state exactly which phenotypes will be selected and give their expected frequencies.
  3. Is there more than one way to obtain the desired genotype ? Which is the best way ? (Assume independent assortment of the three gene pairs.)

## Solution 4: Effective crossing

1. Begin with any two of the three lines and cross them. If, for example, you began with a/a; B/B; C/C x A/A; b/b; C/C, all the progeny would be A/a; B/b; C/C. Crossing two of them would yield:

$$
\begin{aligned}
& 9  & A/\_; B/\_; C/C \\
& 3  & a/a; B/\_; C/C \\
& 3  & A/\_; b/b; C/C \\
& 1  & a/a; b/b; C/C
\end{aligned}
$$

##

\renewcommand{\arraystretch}{2}

```{r gamete-three-combination}
# gamete types
gamete_types <- expand.grid(c("A", "a"), 
                            c("B", "b"), 
                            c("C", "c")) %>% 
  unite(col = "gamete_types", sep = "") %>% 
  pull("gamete_types")

# random mating of gametes
combination_genotypes <- crossing(p2_gamete = gamete_types, p1_gamete = gamete_types) %>% 
  # group_by(gamete_types)
  unite(col = "genotype", remove = FALSE, sep = "") %>% 
  spread(key = p1_gamete, value = genotype)

genotype_orderer <- function(x){
  map_chr(x, function(y){
    str_extract_all(y, "") %>% 
      unlist() %>% 
      str_sort(decreasing = FALSE) %>% 
      paste0(collapse = "")
  })
}

combination_genotypes <- combination_genotypes %>% 
  mutate_all(list(~genotype_orderer(.)))

# # number of unique genotypes
# ca <- c("AA", "Aa", "aa")
# cb <- c("BB", "Bb", "bb")
# cc <- c("CC", "Cc", "cc")
# unique_genotypes <- expand.grid(ca, cb, cc) %>% 
#   unite(col = "all_gametes", sep = "") %>% 
#   pull(all_gametes) %>% 
#   unique()

# manually specify colors
seven_colors <- c("#ffe6ff", "#1a1aff", "#cc3399", "#4d004d", "#ffb31a", "#00994d", "#80ffff")

fanta <- function(x)(stringi::stri_extract_all_regex(x, ".") %>%
                       map(unlist) %>%
                       map_int(~str_detect(.x, "[A-Z]") %>%
                                 sum()))
combination_genotypes %>%
  rename("Gamete types" = p2_gamete) %>% 
  mutate_at(2:9, list(~case_when(
    fanta(.) == 0 ~ kableExtra::cell_spec(., "latex", background = seven_colors[1]),
    fanta(.) == 1 ~ kableExtra::cell_spec(., "latex", background = seven_colors[2]),
    fanta(.) == 2 ~ kableExtra::cell_spec(., "latex", background = seven_colors[3]),
    fanta(.) == 3 ~ kableExtra::cell_spec(., "latex", background = seven_colors[4]),
    fanta(.) == 4 ~ kableExtra::cell_spec(., "latex", background = seven_colors[5]),
    fanta(.) == 5 ~ kableExtra::cell_spec(., "latex", background = seven_colors[6]),
    fanta(.) == 6 ~ kableExtra::cell_spec(., "latex", background = seven_colors[7]),
    TRUE ~ .
  ))) %>%
  kable(booktabs = TRUE, format = "latex", escape = FALSE, linesep = "") %>%
  kableExtra::kable_styling(latex_options = "scale_down", font_size = 8)

```

\renewcommand{\arraystretch}{1}

##

- The a/a; b/b; C/C genotype has two of the gens in a homozygous recessive state and is found in 1/16 of the offspring. 
2. If that phenotype were crossed with A/A; B/B; c/c, all the progeny would be heterozygous for all loci. Crossing two (selfing) of thus obtained individuals would lead to a 27:9:9:9:3:3:3:1 ratio and 1/64 of the progeny would be the desirable a/a; b/b; c/c.
3. There are several different routes to obtaining a/a; b/b; c/c, but the one just outlined only requires four crosses.

## Problem 5: Maternal inheritance and gene assortment

- A man is brachydactylous (very short fingers; rare autosomal dominant), and his wife is not. Both can taste the chemical phenylthiocarbamide (autosomal dominant; common allele), but their mothers could not.
  a. Give the genotypes of the couple. 
If the genes assort independently and the couple has four children, what is the probability of
  b. all of them being brachydactylous?
  c. none being brachydactylous?
  d. all of them being tasters?
  e. all of them being nontasters?
  f. all of them being brachydactylous tasters?
  g. none being brachydactylous tasters?
  h. at least one being a brachydactylous taster?

## Solution 5: Maternal inheritance and gene assortment

a. Let B = brachydactylous, b = normal, T = taster, and t = non-taster. The genotypes of the couple are B/b; T/t for the male and b/b; T/t for the female.
b. For all four children to be brachydactylous, p = $(1/2)^4$ = 1/16.
c. For none of the four children to be brachydactlyous, p = $(1/2)^4$ = 1/16.
d. For all to be tasters, p = $(3/4)^4$ = 81/256
e. For all to be nontasters, p = $(1/4)^4$ = 1/256
f. For all to be brachydactylous tasters, p = $(1/2 x 3/4)^4$ = 81/4096
g. The probability of not being a brachydactylous taster is 1-(the probability of being a brachydactylous taster), or 1-(1/2 x 3/4) = 5/8. The probability that all four children are not brachydactylous tasters is $(5/8)^4$ = 625/4096
h. The probability that at least one is a brachydactylous taster is 1-(probability of none being a brachydactlyous taster), or $1-(5/8)^4$

## Problem 6: Polygene doses and phenotypic classes

- Question: In polygenic systems, how many phenotypic classes corresponding to number of polygene "doses" are expected in selfs.
  a. of strains with four heterozygous polygenes ?
  b. of strains with six heterozygous polygenes ?

## Solution 6: Polygene doses and phenotypic classes

a. $\longrightarrow$ There should be nine classes corresponding to 0, 1, 2, 3, 4, 5, 6, 7, 8 "doses". Table below ( \ref{tab:gamete-four-combination}) shows genotypic composition of individual progenies with distinct phenotypic classess indicated by color coding.
b. $\longrightarrow$ There should be 13 classes corresponding to 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 'doses'.
  
##

\renewcommand{\arraystretch}{2}

```{r gamete-four-combination}
# total possible genotypes
four_genes <- crossing(homolog1 = c("AA", "aa", "Aa") %>%
           combn(3, simplify = TRUE),
         homolog2 = c("BB", "bb", "Bb") %>%
           combn(3, simplify = TRUE),
         homolog3 = c("CC", "cc", "Cc") %>%
           combn(3, simplify = TRUE),
         homolog4 = c("DD", "dd", "Dd") %>%
           combn(3, simplify = TRUE)
) %>%
  unite(col = "paired_homologs", sep = "") %>%
  pull("paired_homologs") %>%
  rev() %>%
  matrix(ncol = 9) %>%
  as_tibble()

# gamete types
gamete_types <- expand.grid(c("A", "a"), 
                            c("B", "b"), 
                            c("C", "c"),
                            c("D", "d")) %>% 
  unite(col = "gamete_types", sep = "") %>% 
  pull("gamete_types")

# random mating of gametes
combination_genotypes <- crossing(p1_gamete=gamete_types, p2_gamete=gamete_types) %>% 
  # group_by(gamete_types)
  unite(col = "genotype", remove = FALSE, sep = "") %>% 
  spread(key = p1_gamete, value = genotype)

genotype_orderer <- function(x){
  map_chr(x, function(y){
    str_extract_all(y, "") %>% 
      unlist() %>% 
      str_sort(decreasing = FALSE) %>% 
      paste0(collapse = "")
  })
}

combination_genotypes <- combination_genotypes %>% 
  mutate_all(list(~genotype_orderer(.)))

# # interpolate automatically
# c(colorRamps::green2red(4), colorRamps::blue2green(5)) %>% colorspace::deutan(3)
# colorRampPalette(c("blue", "red", "white", "green"))(9)
# sample(colors()[grep('gr(a|e)y', colors(), invert = T)], 9)
# viridis::viridis_pal(option = "D")(9)
# c(colorRamps::green2red(4), colorRamps::blue2green(5)) %>% colorspace::deutan(3)
# 
# colorspace::swatchplot(c(colorRamps::green2red(4), colorRamps::blue2green(5)) %>% colorspace::deutan(3))
# colorspace::swatchplot(sample(colors()[grep('gr(a|e)y', colors(), invert = T)], 9))
# colorspace::swatchplot(colorRampPalette(c("blue", "red", "white", "green"))(9))
# colorspace::swatchplot(c("#ffe6ff", "#1a1aff", "#cc3399", "#4d004d", "#ffb31a", "#00994d", "#80ffff", "#ffff1a", "#ff3300"))

# manually specify colors
nine_colors <- c("#ffe6ff", "#1a1aff", "#cc3399", "#4d004d", "#ffb31a", "#00994d", "#80ffff", "#ffff1a", "#ff3300")

fanta <- function(x)(stringi::stri_extract_all_regex(x, ".") %>%
                       map(unlist) %>%
                       map_int(~str_detect(.x, "[A-Z]") %>%
                                 sum()))
four_genes %>%
  mutate_all(list(~case_when(
    fanta(.) == 0 ~ kableExtra::cell_spec(., "latex", background = nine_colors[1]),
    fanta(.) == 1 ~ kableExtra::cell_spec(., "latex", background = nine_colors[2]),
    fanta(.) == 2 ~ kableExtra::cell_spec(., "latex", background = nine_colors[3]),
    fanta(.) == 3 ~ kableExtra::cell_spec(., "latex", background = nine_colors[4]),
    fanta(.) == 4 ~ kableExtra::cell_spec(., "latex", background = nine_colors[5]),
    fanta(.) == 5 ~ kableExtra::cell_spec(., "latex", background = nine_colors[6]),
    fanta(.) == 6 ~ kableExtra::cell_spec(., "latex", background = nine_colors[7]),
    fanta(.) == 7 ~ kableExtra::cell_spec(., "latex", background = nine_colors[8]),
    fanta(.) == 8 ~ kableExtra::cell_spec(., "latex", background = nine_colors[9]),
    TRUE ~ .
  ))) %>%
  knitr::kable(booktabs = TRUE, format = "latex", escape = FALSE, linesep = "", caption = "Polygene involving four genes and phenotype") %>%
  kableExtra::kable_styling(latex_options = "scale_down", font_size = 6)

# expand.grid(c("A", "a"), c("B", "b"), c("C", "c"), c("D", "d")) %>% 
#   as_tibble() %>% 
#   unite(col = "combined", sep = "") %>% 
#   bind_cols(
#     expand.grid(c("a", "A"), c("b", "B"), c("c", "C"), c("d", "D")) %>% 
#       as_tibble() %>% 
#       unite(col = "combined", sep = "")
#   ) %>% 
#   cross_df() %>% 
#   unite(col = "combined", sep = "")

# four_genes %>%
#   kable(booktabs = TRUE, caption = "F2 genotypes resulting from a cross between parents differing for four pair of genes") %>% 
#   kableExtra::kable_styling(font_size = 5)
```
