\PassOptionsToPackage{unicode=true}{hyperref} % options for packages loaded elsewhere
\PassOptionsToPackage{hyphens}{url}
\documentclass[11pt,ignorenonframetext,aspectratio=169]{beamer}
\IfFileExists{pgfpages.sty}{\usepackage{pgfpages}}{}
\setbeamertemplate{caption}[numbered]
\setbeamertemplate{caption label separator}{: }
\setbeamercolor{caption name}{fg=normal text.fg}
\beamertemplatenavigationsymbolsempty
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
\fi
\defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}







\fi

  \usetheme[]{metropolis}






% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
  \usepackage{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}


\newif\ifbibliography


\hypersetup{
      pdftitle={Nucleic acids- DNA, RNA; replication, transcription, translation and genetic code},
        pdfauthor={Deependra Dhakal},
          pdfborder={0 0 0},
    breaklinks=true}
%\urlstyle{same}  % Use monospace font for urls







% Prevent slide breaks in the middle of a paragraph:
\widowpenalties 1 10000
\raggedbottom

  \AtBeginPart{
    \let\insertpartnumber\relax
    \let\partname\relax
    \frame{\partpage}
  }
  \AtBeginSection{
    \ifbibliography
    \else
      \let\insertsectionnumber\relax
      \let\sectionname\relax
      \frame{\sectionpage}
    \fi
  }
  \AtBeginSubsection{
    \let\insertsubsectionnumber\relax
    \let\subsectionname\relax
    \frame{\subsectionpage}
  }



\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

  \setcounter{secnumdepth}{0}


  \usepackage{setspace}
  \usepackage{wasysym}
  % \usepackage{fontenc}
  \usepackage{booktabs,siunitx}
  \usepackage{longtable}
  \usepackage{array}
  \usepackage{multirow}
  \usepackage{wrapfig}
  \usepackage{float}
  \usepackage{colortbl}
  \usepackage{pdflscape}
  \usepackage{tabu}
  \usepackage{threeparttable}
  \usepackage{threeparttablex}
  \usepackage[normalem]{ulem}
  \usepackage{makecell}
  \usepackage{xcolor}
  \usepackage{tikz} % required for image opacity change
  \usepackage[absolute,overlay]{textpos} % for text formatting
  \usepackage[skip=0.333\baselineskip]{caption}
  % \usepackage{newtxtext,newtxmath}% better than txfonts   
  
  \sisetup{per-mode=symbol}
  
  % % Added by CII
  % \usepackage[format=hang,labelfont=bf,margin=0.5cm,justification=centering]{caption}
  % \captionsetup{font=small,width=0.9\linewidth,labelfont=small,textfont={small}}
  % % End of CII addition
  
  \usepackage{subcaption}
  \newcommand{\subfloat}[2][need a sub-caption]{\subcaptionbox{#1}{#2}}
  
  \captionsetup[sub]{font=footnotesize,labelfont=footnotesize,textfont=footnotesize}
  % \captionsetup[subfigure]{font=small,labelfont=small,textfont=small}
  % \captionsetup[subfloat]{font=scriptsize,labelfont=scriptsize,textfont=scriptsize}
  
  % this font option is amenable for beamer, although these are global settings
  \setbeamerfont{caption}{size=\tiny}
  % \setbeamerfont{subcaption}{size=\tiny} % this does not chage subfloat fonts
  % \setbeamerfont{subfloat}{size=\tiny} % this does not change subfloat fonts
   
   % use single line spacing ?
  \singlespacing

  \title[]{Nucleic acids- DNA, RNA; replication, transcription, translation and
genetic code}


  \author[
        Deependra Dhakal
    ]{Deependra Dhakal}

  \institute[
    ]{
    Gokuleshwor Agriculture and Animal Science College\\
Tribhuwan University\\
\textit{ddhakal.rookie@gmail.com}\\
\url{https://rookie.rbind.io}
    }

\date[
      Academic year 2019-2020
  ]{
      Academic year 2019-2020
        }

\begin{document}

% Hide progress bar and footline on titlepage
  \begin{frame}[plain]
  \titlepage
  \end{frame}



\hypertarget{structure-of-dna}{%
\section{Structure of DNA}\label{structure-of-dna}}

\begin{frame}{}
\protect\hypertarget{section}{}

\begin{itemize}
\tightlist
\item
  DNA is the universal, hereditary material (except in certain viruses
  -- RNA viruses).
\item
  The structure of the DNA molecule is a double helix (Figure
  \ref{fig:double-helix}).
\item
  The key features about the DNA molecule are:

  \begin{itemize}
  \tightlist
  \item
    It consists of two polynucleotide chains coiled around a central
    axis in a spiral fashion. The most common natural form of DNA is a
    right-handed double helix of diameter 2.0 nm, called the B-DNA. A
    left-handed form (Z-DNA) and an A-form of DNA also occur under
    certain conditions.
  \item
    The polynucleotide chains are antiparallel; one chain runs in the
    \(5^\prime\) to \(3^\prime\) orientation and the other \(3^\prime\)
    to \(5^\prime\) (carbon atoms of a sugar are conventionally numbered
    from the end closest to the aldehyde or ketone).
  \item
    The two bases in each base pair lie in the same plane. Each plane is
    perpendicular to the axis of the helix. There are 10 base pairs per
    helical turn.
  \item
    The helix has two kinds of alternating external grooves; a deep
    groove (called the major groove) and a shallow groove (called the
    minor groove).
  \item
    The nitrogenous bases on one strand pair with those on the other
    strand in complementary fashion (A always pairs with T, while G
    pairs with C).
  \end{itemize}
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-1}{}

\begin{figure}
\includegraphics[width=0.65\linewidth]{../images/double_helix_dna} \caption{The Watson-Crick model of the DNA double helix (B-form)}\label{fig:double-helix}
\end{figure}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-2}{}

\begin{figure}
\includegraphics[width=0.65\linewidth]{../images/complementary_base_pairing} \caption{Specific base pairing by the formation of hydrogen bonds between two antiparallel DNA strands. The molecular structure of the bases only allows the pairings AT and GC. An AT pair forms two hydrogen bonds; a GC pair forms three hydrogen bonds}\label{fig:base-pairing}
\end{figure}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-3}{}

\begin{itemize}
\tightlist
\item
  In addition to these features, nature of code has following
  characteristics:

  \begin{itemize}
  \tightlist
  \item
    Complementary base pairing means that the replicate of each strand
    is given the base sequence of its complementary strand when DNA
    replicates.
  \item
    Because the strands are \textbf{antiparallel}, when two nucleotides
    are paired the sugar portions of these molecules lie in opposite
    directions (one upward and the other downward along the chain).
  \item
    Because the strands are antiparallel, the convention for writing the
    sequence of bases in a strand is to start from the
    \(\mathrm{5^{\prime}-P}\) terminus at the left (e.g., GAC refers to
    a trinucleotide \(\mathrm{5^\prime-P^{\prime}-GAC-3^\prime-OH}\)).
  \item
    The conventional way of expressing the base composition of an
    organism is by the percentage of \(\text{[G] + [C]}\). This value is
    approximately 50\% for most eukaryotes with only minor variations
    among species. In simpler organisms, there are significant
    variations (e.g., 27\% for \emph{Clostridium}, 50\% for \emph{E.
    coli}, and 76\% for \emph{Sarcina}, all of these organisms being
    bacteria).
  \item
    The chains of the double helix are held together by hydrogen bonds
    between base pairs in opposite strands. The bond between A and T is
    a double bond, while the bond between G and C is a triple hydrogen
    bond.
  \end{itemize}
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-4}{}

\begin{figure}
  \begin{columns}[T,onlytextwidth]

  \column{.36\linewidth}
  \begin{center}
  \includegraphics[width=0.90\linewidth]{../images/nucleotide_blocks1.png}
  \end{center}
  
  \column{.36\linewidth}
  \begin{center}
  \includegraphics[width=0.80\linewidth]{../images/nucleotide_blocks.png}
  \end{center}
  
  \column{.28\linewidth}
  \caption{\newline Nucleotides are made up of three building blocks: a pyrimidine or purine base, a pentose, and phosphoric acid. The base is bonded to the pentose by an N-$\beta$-glycoside, either via N1 of the pyrimidine or via N9 of the purine. The phosphoric acid forms an ester with the primary alcohol group of the pentose. Up to two further phosphoric acid groups can be attached by anhydride bonds to the a phosphoric acid group. The glycoside made up of a base and a ribose is called a nucleoside, and that made up of a base and 2-deoxyribose is called a deoxynucleoside. Nucleotides can be nucleoside monophosphates, diphosphates, or triphosphates, as shown in the example using adenosine and its nucleotides. Deoxynucleotides are thus deoxynucleoside monophosphates, diphosphates, or triphosphates. The sugar in ribonucleic acid (RNA) is ribose, and in deoxyribonucleic acid (DNA) it is 2-deoxyribose. The carbon atoms of the sugar are indicated in the nucleosides and nucleotides (1, 2, 5,...). C1 forms the glycosidic bond. To simplify the chemical skeletal formula, the hydrogens attached to carbons are often omitted in complex formulas (e.g., rings).}
  \label{fig:nucleotide-blocks}
  
  \end{columns}
\end{figure}

\end{frame}

\hypertarget{replication-of-dna}{%
\section{Replication of DNA}\label{replication-of-dna}}

\begin{frame}{}
\protect\hypertarget{section-5}{}

\begin{itemize}
\tightlist
\item
  The DNA double helix strands are so arranged in positive-negative
  relationship that identical duplication or replication of heriditary
  information is possible.
\item
  Two strands separate and a base-complementary partner strand is newly
  formed with each strand. This mode of duplication is called
  semi-conservative mode.
\end{itemize}

\begin{center}\includegraphics[width=0.52\linewidth]{../images/semi_conservative_replication} \end{center}

\end{frame}

\begin{frame}{Steps in replication}
\protect\hypertarget{steps-in-replication}{}

\begin{itemize}
\tightlist
\item
  Unwinding of DNA double helix by \textbf{hellicase}
\item
  Prevention of knotting by relaxation enzyme (\textbf{topoisomerase I})
\item
  Two strands of a replication fork (one \(3^{\prime}\) and the other
  \(5^{\prime}\)) need to be extended by different mechanisms because
  DNA polymerases can only extend the chain at former orientation. Only
  \(3^\prime\) end is continually extended.
\item
  Ligases are able to link \(3^\prime\) ends with \(5^\prime\) ends.
  Without these lagging strand sequences cannot be bonded and can be
  isolated as \textbf{Okazaki fragments}.
\item
  DNA polymerase require a \textbf{primer} (RNA sequences) in order to
  start synthesis. These primers supply the \(3^\prime\) end on which
  the DNA polymerase can build further.
\item
  A sequence region replicated by a replicator is called a
  \textbf{replicon}.
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-6}{}

\begin{center}\includegraphics[width=0.45\linewidth]{../images/dna_replication_process} \end{center}

\end{frame}

\hypertarget{structure-of-rna}{%
\section{Structure of RNA}\label{structure-of-rna}}

\begin{frame}{Background}
\protect\hypertarget{background}{}

\begin{itemize}
\tightlist
\item
  Ribonucleic acid (RNA) is similar in structure to DNA. However, there
  are significant differences, the key ones being:

  \begin{itemize}
  \tightlist
  \item
    RNA consists of ribose sugar (in place of deoxyribose) and uracil in
    place of thymine.
  \item
    Most RNA is predominantly single stranded (except in some viruses).
    Sometimes, the molecule folds back on itself to form double-stranded
    regions. Certain animal and plant viruses use RNA as their genetic
    material.
  \item
    A typical cell contains about ten times more RNA than DNA.
  \item
    Whereas DNA stores genetic information, RNA most often functions in
    the expression of the genetic information.
  \item
    There are three major classes of RNA known to be involved in gene
    expression: ribosomal RNA (rRNA), messenger RNA (mRNA), and transfer
    RNA (tRNA). The site of protein synthesis, the ribosome, contains
    rRNA.
  \end{itemize}
\end{itemize}

\end{frame}

\begin{frame}{mRNA}
\protect\hypertarget{mrna}{}

\begin{itemize}
\tightlist
\item
  Messenger RNA (mRNA) is the molecular carrier of genetic information
  from the DNA to ribosomes, where this DNA transcript or template is
  translated (the genetic information of DNA transcript is expressed)
  into proteins.
\item
  Because genes vary in size (number of nucleotides) the mRNA species
  are variable in length.
\item
  Eukaryotic RNA is transcribed as a primary transcript, a cap is added
  at the \(5^\prime\) end, a poly(A) tail is added, and the introns are
  removed.
\item
  The mRNA is then transported from the nucleus to the endoplasmic
  reticulum for translation by ribosomes.
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-7}{}

\begin{figure}
\includegraphics[width=0.65\linewidth]{../images/mrna_processed} \caption{Detailed structure of a processed eukaryotic mRNA. The cap structure is followed by $5^\prime$ UTR, the protein encoding exongs, $3^\prime$ UTR, and a poly(A) tail.}\label{fig:mrna-processed}
\end{figure}

\end{frame}

\begin{frame}{tRNA}
\protect\hypertarget{trna}{}

\begin{itemize}
\tightlist
\item
  The structure of transfer RNA (tRNA) is very unique among the three
  key RNA molecules in the cell.
\item
  These molecules are small in size and very stable (range in size from
  75--90 nucleotides).
\item
  A single-stranded tRNA molecule is able to fold back onto itself and
  undergo complementary base pairing in short stretches to form double
  strands.
\item
  This folding also creates four characteristic loops and a cloverleaf
  2-D structure (Figure \ref{fig:ribonucleic-acids}).

  \begin{itemize}
  \tightlist
  \item
    Three loops are involved in translating the message of the mRNA.
  \item
    The anticodon loop (or simply anticodon) consists of a sequence of
    three bases that are complementary to the sequence of a codon on the
    mRNA.
  \end{itemize}
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-8}{}

\begin{itemize}
\tightlist
\item
  The stop codons do not have tRNA with anticodons for them.
\item
  Another feature of the tRNA molecule is the occurrence of the sequence
  pCpCpA- \(3^\prime\) at the \(3^\prime\) end.
\item
  The terminal adenine residue is the point of attachment for an amino
  acid and hence is called the amino acid attachment (or binding) site.
\item
  During protein synthesis, the amino acid corresponding to a particular
  mRNA codon that base pairs with the tRNA anticodon is attached to this
  terminal and transported to the appropriate segment of the mRNA.
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-9}{}

\begin{itemize}
\tightlist
\item
  Ribosomes are the sites (``factories'') of polypeptide synthesis (or
  protein synthesis).
\item
  A bacterial cell may contain about 1000 ribosomes.
\item
  A ribosome consists of two subunits, which together form the monosome.
  The ribosomal particles are classified according to their
  sedimentation coefficient or rate (S).
\item
  Monosomes of bacteria are 70S (70S ribosomes) whereas eukaryotic
  monosomes are about 80S.
\item
  Because sedimentation coefficients are not additive, a 70S monosome in
  actuality comprises two subunits that are 50S and 30S, while an 80S
  monosome consists of 60S and 40S subunits.
\item
  A ribosome subunit consists of molecules of rRNA and proteins.
\item
  For example, the 50S subunit contains one 55 rRNA, one 235 rRNA
  molecule, and 32 different ribosomal proteins.
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-10}{}

\begin{figure}
  \begin{columns}[T,onlytextwidth]

  \column{.65\linewidth}
  
  \begin{columns}[T,onlytextwidth]
  \column{.55\linewidth}  
  \begin{center}
  \includegraphics[width=0.95\linewidth]{../images/ribonucleic_acids_trna.png}
  \end{center}
  
  \column{.45\linewidth}
  \begin{center}
  \includegraphics[width=0.95\linewidth]{../images/ribonucleic_acids_types.png}
  \end{center}
  \end{columns}
  
  \begin{center}
  \includegraphics[width=0.95\linewidth]{../images/ribonucleic_acids_viral_molecule.png}
  \end{center}

  \column{.35\linewidth}
  \caption{\newline Ribonucleic acids. (a) 'Clover-leaf' structure of a transfer RNA (tRNA) molecule with the tyrosine-bearing tRNA ($tRNA^{Tyr}$) of yeast. The structure is stabilized by regions inside the single-stranded molecule that can swap base pairs; there are adjacent unpaired regions. Numerous bases of tRNAs (outlined in red) are modified; some important examples of modified bases are shown in (b). Adjacent there are methylated bases ($Me$). A base triplet, the anticodon, pairs with a complementary base triplet on the mRNA. The codon–anticodon sequence of triplets on the mRNA specifies the order of the amino acids in a protein. The D arm is named after the increased frequency of dihydrouracil ($UH_2$), and the TCC arm is named after the ubiquitous base sequence $5^{\prime} - T - \Psi - C - 3^{\prime}$. The V loop differs in size in different tRNAs. The amino acid acceptor arm and the anticodon arm are important for the recognition of the right tRNAs and their amino acids by the aminoacyl-tRNA-synthetases. The anticodon arm 'presents' the anticodon simultaneously with the mRNA codon so that base pairing can occur on the ribosome. The $\mathrm{T\Psi C}$ arm and the D arm are probably especially significant for the bonding of the tRNA to the ribosome. (c) Example of a ring-shaped closed RNA molecule. The potato spindle tuber viroid consists of a covalently closed ring of 359 nucleotides. The structure is stabilized by intramolecular base pairs, represented here as simple lines (for clarity)}
  \label{fig:ribonucleic-acids}

  \end{columns}
\end{figure}

\end{frame}

\begin{frame}{Central dogma of biology}
\protect\hypertarget{central-dogma-of-biology}{}

\begin{itemize}
\tightlist
\item
  The genetic information of the DNA is changed into biological material
  principally through proteins, according to the central dogma of
  molecular biology.
\item
  The dogma states that genetic information flow is generally
  unidirectional from DNA to proteins, except in special cases (Figure
  ).
\item
  This flow, mediated by transcription (copying of the DNA template by
  synthesizing the RNA molecule) and translation (synthesis of a
  polypeptide using the genetic information encoded in an mRNA molecule)
  and preceded by replication (the process of DNA synthesis), can now be
  reversed in vitro (in the test tube) by scientists.
\item
  Thus, once a protein is known, the nucleotide sequence in the
  prescribing DNA strand can be determined and synthesized (the product
  is called a complementary DNA or cDNA). Production of cDNA is a
  technique used in genetic engineering.
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-11}{}

\begin{figure}
\includegraphics[width=0.45\linewidth]{../images/central_dogma_biology} \caption{The central dogma of molecular genetics, showing information flow involving DNA, RNA and proteins within a cell.}\label{fig:central-dogma-biology}
\end{figure}

\end{frame}

\hypertarget{expression-of-genetic-information}{%
\section{Expression of genetic
information}\label{expression-of-genetic-information}}

\begin{frame}{Genetic code}
\protect\hypertarget{genetic-code}{}

\begin{itemize}
\tightlist
\item
  A key question in genetics is how the information of the DNA is
  interpreted to produce protein.
\item
  The sequence of bases in the polynucleotide chain holds the key to DNA
  function because it represents the genetic code (the set of rules
  giving the correspondence between mRNA and amino acids in a protein)
  for the synthesis of corresponding amino acids that constitute
  proteins.
\item
  DNA does not code for adult traits directly, there being no genes for
  adult traits as such. Instead, genes code for various developmental
  processes.
\item
  The variety of protein products in a cell undertake catalytic and
  structural activities that eventually result in an adult phenotype.
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-12}{}

\begin{itemize}
\tightlist
\item
  There are about twenty commonly occurring amino acids.
\item
  According to the prescribed sequence (based on the genetic code),
  amino acids are joined together by \textbf{peptide bonds} to form
  polypeptide chains (Figure \ref{fig:polypeptides-amino}).
\item
  The genetic code is a triplet code.
\item
  Three adjacent bases form a code for an amino acid.
\item
  Each trinucleotide sequence is called a codon (Figure
  \ref{fig:universality}).
\item
  The genetic code is read from a fixed starting point of the DNA
  strand.
\item
  The genetic code is said to be degenerate because nearly all amino
  acids are specified by at least two codons.
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-13}{}

\begin{figure}
\includegraphics[width=0.45\linewidth]{./../images/nucleotide_codons} \caption{The genetic code is universal for all life forms}\label{fig:universality}
\end{figure}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-14}{}

\begin{itemize}
\tightlist
\item
  Some (serine, arginine, leucine) are encoded by six different codons.
\item
  Only tryptophan and methionine are encoded by single codons.
\item
  Further, for a set of codons encoding the same amino acid, the first
  two letters in the figure is the same, with only the third being
  different (called the \textbf{wobble hypothesis}).
\item
  Consequently, at least 30 different tRNA species are required to
  account for the 61 different triplets in the coding dictionary in just
  17 (the three remaining triplets include termination codons or signals
  -- UAG, UAA, UGA).
\end{itemize}

\end{frame}

\begin{frame}{Transcription: RNA synthesis}
\protect\hypertarget{transcription-rna-synthesis}{}

\begin{itemize}
\tightlist
\item
  The genetic information of the DNA template is copied by the process
  of transcription (or RNA synthesis) to produce an RNA sequence (mRNA).
\item
  The transactions of DNA and RNA take place through the matching of
  complementary bases and the binding of various proteins to specific
  sites on the DNA or RNA.
\item
  The DNA strand that is transcribed is called the \textbf{template
  strand}.
\item
  The process starts with a recognition of a special DNA sequence
  (called a promoter) and binding to it by an enzyme, a process called
  template binding.
\item
  RNA polymerase II carries out the transcription in eukaryotes.
\item
  Transcription factors control its activity.
\item
  The RNA chain then grows (chain elongation) in the \(3^\prime\)
  direction (Starting at \(5^\prime\)).
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-15}{}

\begin{itemize}
\tightlist
\item
  Binding and activity of RNA polymerase II are controlled by
  cis-regulatory regions and by trans-acting regulatory factors
  (transcription factors).
\item
  General transcription factors (GTFs) are necessary for RNAP II to
  transcribe DNA, and Specific TFs affect the efficiency or rate of RNAP
  II transcription.
\item
  Promotor region is located at a gene's \(5^\prime\) end (using the
  coding strand as a reference) -- upstream from transcription start
  site or the \textbf{ORF}.
\item
  Promoter is composed of core promoter and proximal promotor elements,
  that define when and where a gene is transcribed.
\item
  Nucleotide sequences are numerically designated based on transcription
  initiation site as reference point (``+'' ve and ``-'' ve, former
  referring to downstream sequences, and latter to the upstream
  sequences)
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-16}{}

\begin{figure}
\includegraphics[width=0.65\linewidth]{../images/gene_transcription} \caption{(a) The promoter lies 'upstream' (toward the $5^\prime$ end) of the initiation point and coding sequences. (b) Promoters have regions of similar sequences, as indicated by the yellow shading in seven different promoter sequences in \textit{E. coli}. Spaces (dots) are inserted in the sequences to optimize the alignment of the common sequences. Numbers refer to the number of bases before (-) or after (+) the RNA synthesis initiation point. The consensus sequence for most E. coli promoters is at the bottom.}\label{fig:gene-transcription}
\end{figure}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-17}{}

\begin{itemize}
\tightlist
\item
  An RNA polymerase holoenzyme binds to the DNA at \textbf{consensus
  sequence}, then unwinds the DNA double helix and begins the synthesis
  of an RNA molecule.
\item
  RNA polymerase holoenzyme is a multisubunit complex is composed of the
  five subunits of the core enzyme (two subunits of a, one of \(\beta\),
  one of \(\beta^\prime\), and one of \(\omega\)) + a subunit called
  sigma (\(\sigma\)) factor.
\item
  Although ATG (AUG in the mRNA) sequence is the initiation codon
  translated as protein, mRNA sequence consists of portion of the genes
  starting from +1 site, which are well upstream of the AUG sequence.
\item
  The intervening part is referred to as \(5^\prime\) untranslated (UTR)
  region. Similarly occurs a sequence to the \(3^\prime\) region of the
  transcribed RNA, called \(3^\prime\) UTR region.
\item
  As the RNA polymerase moves along the DNA, it unwinds the DNA ahead of
  it and rewinds the DNA that has already been transcribed. In this way,
  it maintains a region of single-stranded DNA, called a
  \textbf{transcription bubble}.
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-18}{}

\begin{columns}[T,onlytextwidth]

  \column{.5\linewidth}

\begin{figure}
\includegraphics[width=0.8\linewidth]{../images/gene_transcription_prokaryotes1} \caption{(a-b) In transcription initiation, $\sigma$ subunit positions prokaryotic RNA polymerase for transcription initiation}\label{fig:gene-transcription-prokaryotes1}
\end{figure}

  \column{.5\linewidth}
  
\begin{figure}
\includegraphics[width=0.75\linewidth]{../images/gene_transcription_prokaryotes2} \caption{(a) Elongation: Synthesis of an RNA strand complementary to the single-strand region of the DNA template strand. DNA that is unwound ahead of RNA polymerase is rewound after it has been transcribed.}\label{fig:gene-transcription-prokaryotes2}
\end{figure}


\begin{figure}
\includegraphics[width=0.75\linewidth]{../images/gene_transcription_prokaryotes3} \caption{(b) Termination: The intrinsic mechanism shown here is one of two ways used to end RNA synthesis and release the completed RNA transcript and RNA polymerase from the DNA.}\label{fig:gene-transcription-prokaryotes3}
\end{figure}

\end{columns}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-19}{}

\begin{itemize}
\tightlist
\item
  The first product of transcription in eukaryotes is called a
  \textbf{pre-mRNA}, part of a group of molecules called heterogeneous
  nuclear DNA (hmRNA).
\item
  This molecule undergoes severe alterations to remove non-coding parts
  (\textbf{introns}) of the sequence, leaving the coding parts
  (\textbf{exons}) to produce the mRNA, which is typically about 25\% of
  the original length of the pre-mRNA.
\item
  After removing the introns, the splicing or linking of the pieces
  results in different types of mRNA (called \textbf{alternative
  splicing}). Consequently, different kinds of proteins may be encoded
  by the same gene (Figure \ref{fig:mrna-alternative-splicing}).
\item
  The mRNA is transported to the ribosomes.
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-20}{}

\begin{figure}
\includegraphics[width=0.65\linewidth]{../images/alternative_splicing} \caption{The pre-mRNA transcript of the rat $\alpha$-tropomyosin gene is alternatively spliced in different cell types. The light green boxes represent introns; the other colors represent exons. Polyadenylation signals are indicated by an A. Dashed lines in the mature mRNAs indicate regions that have been removed by splicing.}\label{fig:mrna-alternative-splicing}
\end{figure}

\end{frame}

\begin{frame}{Translation: protein synthesis}
\protect\hypertarget{translation-protein-synthesis}{}

\begin{itemize}
\tightlist
\item
  Protein synthesis consists of three steps-initiation, elongation, and
  termination.
\item
  Translation starts with the formation of an initiation complex that
  includes initiation factors that bind to the small rRNA subunit and
  then to the mRNA.
\item
  The next step is to set the reading frame for accurate translation.
\item
  The AUG triplet is usually the initiation point.
\item
  The large subunit binds to the complex. The sequence of the next
  triplet determines what charged tRNA (has an amino acid attached) will
  be attached.
\item
  The process is repeated until the whole mRNA is translated, adjacent
  amino acids being linked by peptide bonds.
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-21}{}

\begin{itemize}
\tightlist
\item
  The termination of translation occurs when the elongation process
  encounters a stop codon or termination codon.
\item
  The interval between the start and stop codons that encodes an amino
  acid for insertion into a polypeptide chain is called the open reading
  frame (ORF).
\item
  Each gene codes for one polypeptide. Some proteins comprise more than
  one polypeptide (have multiple subunits).
\item
  All genes do not code for proteins and, further, all genes in a cell
  are not actively transcribing mRNA all of the time. Also, most enzymes
  are proteins, but all proteins are not enzymes.
\end{itemize}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-22}{}

\begin{figure}
\includegraphics[width=0.45\linewidth]{../images/polypeptides_amino_acid} \caption{(a) The basic structure of an amino acid consists of three units - an amino group, a carboxyl group, and a side chain (R) that distinguishes among the different amino acids. (b) A polypeptide chain is formed by linking many amino acids together; adjacent amino acids are linked by a peptide bond.}\label{fig:polypeptides-amino}
\end{figure}

\end{frame}

\begin{frame}{}
\protect\hypertarget{section-23}{}

\begin{figure}
\includegraphics[width=0.35\linewidth]{../images/translation_prokaryotes} \caption{(A) Initiation. Initiation of translation begins with the association of the small ribosome subunit with the Shine–Dalgarno sequence (S-D sequence) on the mRNA. Next, initiation factors IF1, IF2, and IF3 (not shown) charge or connect the initiator tRNA with fMet. The charged initiator tRNA ($tRNA^{ifMet}$) associates with the small ribosome subunit and finds the start codon. Finally, the large ribosomal subunit joins the small subunit and situates the initiator tRNA at the P site. (B) Elongation. During elongation, peptide bonds are formed between the amino acids at the A-site and the P-site. The movement of the ribosome along the mRNA and addition of a new tRNA to the A-site are controlled by elongation factors (also not shown). (C) Termination. Termination requires release factors. The various components dissociate. The completed protein folds into its proper three-dimensional shape.}\label{fig:translation-prokaryotes}
\end{figure}

\end{frame}

\hypertarget{bibliography}{%
\section{Bibliography}\label{bibliography}}

\begin{frame}{Also see}
\protect\hypertarget{also-see}{}

\end{frame}

\begin{frame}{References}
\protect\hypertarget{references}{}

\end{frame}




\end{document}
