% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
\PassOptionsToPackage{dvipsnames,svgnames,x11names}{xcolor}
%
\documentclass[
  ignorenonframetext,
  aspectratio=169]{beamer}
\usepackage{pgfpages}
\setbeamertemplate{caption}[numbered]
\setbeamertemplate{caption label separator}{: }
\setbeamercolor{caption name}{fg=normal text.fg}
\beamertemplatenavigationsymbolsempty
% Prevent slide breaks in the middle of a paragraph
\widowpenalties 1 10000
\raggedbottom
\setbeamertemplate{part page}{
  \centering
  \begin{beamercolorbox}[sep=16pt,center]{part title}
    \usebeamerfont{part title}\insertpart\par
  \end{beamercolorbox}
}
\setbeamertemplate{section page}{
  \centering
  \begin{beamercolorbox}[sep=12pt,center]{part title}
    \usebeamerfont{section title}\insertsection\par
  \end{beamercolorbox}
}
\setbeamertemplate{subsection page}{
  \centering
  \begin{beamercolorbox}[sep=8pt,center]{part title}
    \usebeamerfont{subsection title}\insertsubsection\par
  \end{beamercolorbox}
}
\AtBeginPart{
  \frame{\partpage}
}
\AtBeginSection{
  \ifbibliography
  \else
    \frame{\sectionpage}
  \fi
}
\AtBeginSubsection{
  \frame{\subsectionpage}
}
\usepackage{amsmath,amssymb}
\usepackage{lmodern}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
\usetheme[]{Frankfurt}
\usecolortheme{beaver}
\usefonttheme{structuresmallcapsserif}
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\newif\ifbibliography
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\usepackage{setspace}
\usepackage{wasysym}
% \usepackage{fontenc}
\usepackage{booktabs,siunitx}
\usepackage{longtable}
\usepackage{array}
\usepackage{multirow}
\usepackage{wrapfig}
\usepackage{float}
\usepackage{colortbl}
\usepackage{pdflscape}
\usepackage{tabu}
\usepackage{threeparttable}
\usepackage{threeparttablex}
\usepackage[normalem]{ulem}
\usepackage{makecell}
\usepackage{xcolor}
\usepackage{tikz} % required for image opacity change
\usepackage[absolute,overlay]{textpos} % for text formatting
\usepackage[skip=0.333\baselineskip]{caption}
% \usepackage{newtxtext,newtxmath}% better than txfonts

% define new footline defination
\setbeamertemplate{footline}
{
    \leavevmode%
    \hbox{%
        \begin{beamercolorbox}[wd = .5\paperwidth, ht = 1ex, dp = 1ex, center]{author in head/foot}%
            Text in footer
        \end{beamercolorbox}%
        \begin{beamercolorbox}[wd = .5\paperwidth, ht = 1ex, dp = 1ex, center]{date in head/foot}%
            \insertframenumber{}
        \end{beamercolorbox}
    }%
    \vskip3pt%
}
% add footline defination to the beamer template
\addtobeamertemplate{footline}{\begin{center}\rule{0.6\paperwidth}{0.4pt}\end{center}\vspace*{-1ex}}{}

% % set background image if you will
% \usebackgroundtemplate%
% {%
%     \includegraphics[width=\paperwidth,height=\paperheight]{02-dna_modification_background_dna_helix.jpg}%
% }

% % set background in a TikZ node for modifications
\usepackage{tikz}
\usebackgroundtemplate{%
\tikz\node[opacity=0.3] {\includegraphics[height=\paperheight,width=\paperwidth]{02-dna_modification_gene_cloning_rdna_technology_bg_dna_helix.jpg}};}

% % set caption font size
% % note that beamer presentation native captions have their own configs
% \usepackage{caption}
% \captionsetup{font=footnotesize}

% this font option is amenable for beamer
\setbeamerfont{caption}{size=\tiny}

% some beamer themes naturally might not support navigation symbols
% \setbeamertemplate{navigation symbols}{} % remove navigation symbols

\setbeamertemplate{footline}[page number] % insert page number in footline

% \setbeamertemplate{navigation symbols}{slide} % insert slide indication in navigation
% \setbeamertemplate{navigation symbols}{frame} % insert frame indication in navigation
% \setbeamertemplate{navigation symbols}{section} % insert section indication in navigation
% \setbeamertemplate{navigation symbols}{subsection} % insert subsection indication in navigation

% \AtBeginSubsection{} % supress subsection display

\newcommand{\bcolumns}{\begin{columns}[T, onlytextwidth]}
\newcommand{\ecolumns}{\end{columns}}

\newcommand{\bdescription}{\begin{description}}
\newcommand{\edescription}{\end{description}}

\newcommand{\bitemize}{\begin{itemize}}
\newcommand{\eitemize}{\end{itemize}}
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\urlstyle{same} % disable monospaced font for URLs
\hypersetup{
  pdftitle={DNA modification, gene cloning and rDNA technology},
  pdfauthor={Deependra Dhakal},
  colorlinks=true,
  linkcolor={red},
  filecolor={Maroon},
  citecolor={blue},
  urlcolor={red},
  pdfcreator={LaTeX via pandoc}}

\title{DNA modification, gene cloning and rDNA technology}
\author{Deependra Dhakal}
\date{}
\institute{Agriculture and Forestry
University \and \href{mailto:ddhakal.rookie@gmail.com}{\nolinkurl{ddhakal.rookie@gmail.com}} \and \url{https://rookie.rbind.io/}}

\begin{document}
\frame{\titlepage}

\begin{frame}[allowframebreaks]
  \tableofcontents[hideallsubsections]
\end{frame}
\hypertarget{genetic-structure}{%
\section{Genetic Structure}\label{genetic-structure}}

\begin{frame}{History}
\protect\hypertarget{history}{}
\begin{itemize}
\tightlist
\item
  1972, two researchers (Herbert W. Boyer, UCSD and Stanley N. Cohen,
  Stanford) met at a conference in Hawaii to discuss \emph{plasmids}.
\item
  One studied Plasmids
\item
  Other studied restriction and modification enzymes (EcoRI)
\item
  After talks, ideas combined.
\end{itemize}
\end{frame}

\begin{frame}{Structure of DNA}
\protect\hypertarget{structure-of-dna}{}
\begin{figure}
\includegraphics[width=0.45\linewidth]{./../images/dna_structure_a} \caption{DNA has two strands antiparallel to each other. The structure of the subcomponents is shown to the sides.}\label{fig:nucleic-acid-dna}
\end{figure}
\end{frame}

\begin{frame}{Structure of RNA}
\protect\hypertarget{structure-of-rna}{}
\begin{figure}
\includegraphics[width=0.45\linewidth]{./../images/dna_structure_b} \caption{RNA is usually single-stranded and has two chemical differences from DNA. First, an extra hydroxyl group (-OH) is found at the 2 prime position of ribose, and second, thymine is replaced by uracil.}\label{fig:nucleic-acid-rna}
\end{figure}
\end{frame}

\begin{frame}{DNA modification: simplified}
\protect\hypertarget{dna-modification-simplified}{}
\begin{itemize}
\tightlist
\item
  Identification and isolation of the desired gene/DNA fragment to be
  cloned from different animals, other bacteria, and viruses.
  Restriction endonucleases are used for producing desired fragments of
  a given genome.
\item
  Insertion of the gene/DNA segment in a vector; plasmid, virus, cosmid,
  etc.
\item
  Introduction of the recombinant vector into a suitable host
  (transformation).
\item
  Integration of the desired gene into the nuclear/organeller genome of
  the host cell.
\item
  Selection of the transformed host cells.
\item
  Expression of the cloned genes.
\item
  Regeneration of the cloned genes.
\item
  Transmission to the progenies.
\end{itemize}
\end{frame}

\hypertarget{dna-isolation-and-purification}{%
\section{DNA isolation and
purification}\label{dna-isolation-and-purification}}

\begin{frame}{Overview}
\protect\hypertarget{overview}{}
\begin{itemize}
\tightlist
\item
  Cell structure and organelles present a natural barrier
\item
  Bacteria are easiest to manipulate. How?
\item
  Distinction between two DNAs is made based on their sizes.
\end{itemize}
\end{frame}

\begin{frame}{Isolation steps}
\protect\hypertarget{isolation-steps}{}
\begin{enumerate}
\tightlist
\item
  Destruction of cell membrane. (Lysozyme digestion of peptidoglycan in
  bacteria)
\item
  Bursting of cell membrane by destruction of lipid bilayer (by
  detergent such as sodium dodecyl sulfate (SDS)). In plants and
  animals, tissue samples are generally ground up to release the
  intracellular components.
\item
  Separation of intracellular components from the insoluble remains
  (cellular membranes, bones, cartilage, etc.) by either centrifugation
  or chemical extraction.
\item
  Extraction of unwanted proteins from DNA with phenol (Dissolves
  60\%-70\% of all living matter).
\end{enumerate}
\end{frame}

\begin{frame}{Purification procedure}
\protect\hypertarget{purification-procedure}{}
\begin{itemize}
\tightlist
\item
  The two phases (one that of proteins dissolved in phenol and the other
  of nucleic acids in the aqueous layer) are separated by
  centrifugation, and the aqueous DNA layer is removed form the phenol
\item
  Use enzyme ribonuclease (RNase) to digests RNA into small
  ribonucleotide fragments
\item
  Treat this solution with alcohol to precipitate large DNA out of the
  aqueous phase, while ribonucleotides stay soluble, thus favoring
  centrifugation extraction of DNA
\end{itemize}
\end{frame}

\hypertarget{dna-fragment-separation-using-electrophoresis}{%
\section{DNA fragment separation using
Electrophoresis}\label{dna-fragment-separation-using-electrophoresis}}

\begin{frame}{Initial preparation}
\protect\hypertarget{initial-preparation}{}
\begin{itemize}
\tightlist
\item
  Gel electrophoresis is used to separate DNA fragments by size
\item
  Gel consists of agarose, a polysaccharide extracted from seaweed that
  behaves like gelatin (commercial powder).
\item
  For visualizing DNA agarose gel is solidified, after subsequent
  cooling once the powder-water mixture is heated, into a rectangular
  slab about 1/4 inch thick by casting the molten liquid into a special
  tray.
\item
  To make small wells or holes at one end of the gel, comb is inserted
  before the gel hardens.
\end{itemize}
\end{frame}

\begin{frame}{Process}
\protect\hypertarget{process}{}
Gel electrophoresis uses electric current to separate DNA molecules by
size. The agarose slab is immersed in a buffer-filled tank that has a
positive electrode at one end and a negative electrode at the other. DNA
samples are loaded into the wells, and when an electrical field is
applied, the DNA migrates through the gel. The phosphate backbone of DNA
is negatively charged, so it moves away from the negative electrode and
toward the positive electrode. Polymerized agarose acts as a sieve with
small holes between the tangled chains of agarose. The DNA must migrate
through these gaps. Agarose separates the DNA by size because larger
pieces of DNA are slowed down more by the agarose.
\end{frame}

\begin{frame}{Visualization}
\protect\hypertarget{visualization}{}
\begin{itemize}
\tightlist
\item
  The agarose gel is removed from the tank and immersed into a solution
  of ethidium bromide (mutagenic).
\item
  This dye intercalates between the bases of DNA or RNA, although less
  dye binds to RNA because it is single-stranded. When the gel is
  exposed to ultraviolet light, it fluoresces bright orange.
\item
  In Figure \ref{fig:gel-electrophoresis}, the DNA fragments are
  visualized by a positively charged dye from the thiazin family. The
  dye interacts with the negatively charged backbone of the DNA and is a
  nontoxic alternative that does not require ultraviolet light sources.
\end{itemize}
\end{frame}

\begin{frame}{Notes}
\protect\hypertarget{notes}{}
\begin{itemize}
\tightlist
\item
  Size of DNA being examined affects what type of gel is used.
\item
  Size of DNA molecules can be determined by comparing to a set of
  molecular weight standards run in a different well.
\item
  Because the standards are of known size, the experimental DNA fragment
  can be compared directly.
\item
  Resolvability of some gel matrix is different than other.
\end{itemize}

\begin{block}{}
\textrm{Fragments of DNA are separated by size using gel electrophoresis. A current causes the DNA fragments to move away from the negative electrode and toward the positive. As the DNA travels through agarose, the larger fragments get stuck in the gel pores more than the smaller DNA fragments. Pulsed field gel electrophoresis separates large pieces of DNA by alternating the electric current at right angles.}
\end{block}
\end{frame}

\begin{frame}{Electrophoresis apparatus and visualization}
\protect\hypertarget{electrophoresis-apparatus-and-visualization}{}
\begin{figure}
\includegraphics[width=0.45\linewidth]{./../images/electrophoresis} \caption{(A) Photo of electrophoresis supplies. Electrophoresis chamber holds an agarose gel in the center portion, and the rest of the tank is filled with buffer solution. The red and black leads are then attached to an electrical source. FisherBiotech Horizontal Electrophoresis Systems, Midigel System; Standard; 13 $\times$ 16-cm gel size; 800 mL buffer volume; Model No. FB-SB-1316. (B) Agarose gel separation of DNA. The size of the fragments can be calculated by comparing them to the standard DNA marker in lane 1. The brighter bands in the marker are 1000 base pairs and 500 base pairs, with the 1000 base-pair marker closer to the wells (marked with numbers 1–8).}\label{fig:gel-electrophoresis}
\end{figure}
\end{frame}

\hypertarget{dna-modification}{%
\section{DNA modification}\label{dna-modification}}

\begin{frame}{DNA cutters: Restriction enzymes}
\protect\hypertarget{dna-cutters-restriction-enzymes}{}
\begin{itemize}
\tightlist
\item
  DNA should be cut to fragments of different sizes because entire set
  of nuclear DNA is not of particular interest in visualization
\item
  Naturally occuring \emph{restriction enzymes} or \emph{restriction
  endonucleases} come into rescue in this operation.
\item
  These bacterial enzymes bind to specific recognition sites on DNA and
  cut the backbone of both strands.
\item
  These enzymes are methylation sensitive.
\item
  Bacteria produce \emph{modification enzymes} that recognize the same
  sequence as the corresponding restriction enzyme. These methylate each
  recognition site in the bacterial genome.
\end{itemize}
\end{frame}

\begin{frame}{DNA cutters: Restriction enzymes}
\protect\hypertarget{dna-cutters-restriction-enzymes-1}{}
\begin{figure}
\includegraphics[width=0.5\linewidth]{./../images/restriction_enzymes} \caption{\textbf{Type II Restriction Enzymes -- Blunt versus Sticky ends} \newline \textit{Hpal} is a bunt-end restriction enzyme; that is, it cuts both strands of DNA in exactly the same position.    extit{EcoR}I is a sticky-end restriction enzyme. The enzyme cuts between the G and A on both strands, which generates four base-pair overhangs on the ends of the DNA. Since these ends may base pair with complementary sequences, they are considered 'sticky'.}\label{fig:restriction-enzymes}
\end{figure}
\end{frame}

\begin{frame}{DNA cutters: Restriction enzymes}
\protect\hypertarget{dna-cutters-restriction-enzymes-2}{}
\begin{itemize}
\item
  Restriction enzymes have been exploited to cut DNA at specific sites,
  since each restriction enzyme has a particular recognition sequence.
  Difference in cleavage site determine the type of restriction enzyme.

  \begin{itemize}
  \tightlist
  \item
    Type I restriction enzymes cut the DNA strand 1000 or more base
    pairs from the recognition sequence.
  \item
    Type II restriction enzymes cut in the middle of the recognition
    sequence and are the most useful in genetic engineering. The Type II
    restriction enzymes can form \emph{both} sticky or \emph{blunt}
    ends.
  \end{itemize}
\item
  The recognition sequences of Type II restriction enzymes are usually
  inverted repeats so that the enzymes cut between the same bases on the
  both strands.
\end{itemize}
\end{frame}

\begin{frame}{DNA cutters: Restriction enzymes}
\protect\hypertarget{dna-cutters-restriction-enzymes-3}{}
\begin{figure}
\includegraphics[width=0.5\linewidth]{./../images/restriction_enzymes_ex} \caption{Common restriction enzymes}\label{fig:restriction-enzymes-ex}
\end{figure}
\end{frame}

\begin{frame}{DNA cutters: Restriction enzymes}
\protect\hypertarget{dna-cutters-restriction-enzymes-4}{}
\begin{itemize}
\tightlist
\item
  The number of base pairs in the recognition sequence determines the
  likelihood of cutting.
\item
  So to generate fewer, longer fragments, restriction enzymes with six
  or more base-pair recognition sequences are used.
\end{itemize}
\end{frame}

\begin{frame}{DNA joiners: Ligase -- Overview}
\protect\hypertarget{dna-joiners-ligase-overview}{}
\begin{itemize}
\tightlist
\item
  When two different DNA samples are cut with the same sticky-end
  restriction enzyme, all fragments will have identical overhangs.
\item
  This allows DNA fragments from two sources (e.g., two different
  organisms) to be linked together (Figure
  \ref{fig:restriction-ligation}).
\item
  Fragments are linked or ligated using DNA ligase
\item
  The efficiency of ligases depends on the total concentration of the
  substrates and also on the closeness of the ends to be ligated.
\end{itemize}
\end{frame}

\begin{frame}{DNA joiners: Ligase -- Types}
\protect\hypertarget{dna-joiners-ligase-types}{}
\begin{itemize}
\tightlist
\item
  There are 3 kinds of mammalian DNA ligases.
\item
  Joining of Okazaki fragments and sealing of excision repair nicks,
  repair of double-strand breaks and nonhomologous end joining, joining
  of recombining plasmid vectors.
\end{itemize}
\end{frame}

\begin{frame}{DNA joiners: Ligase}
\protect\hypertarget{dna-joiners-ligase}{}
\begin{figure}
\includegraphics[width=0.14\linewidth]{./../images/restriction_ligation} \caption{\textbf{Compatible overhangs are linked using DNA Ligase}\newline BamHI and Bgl Il generate the same overhanging or sticky ends: a $3^\prime$-CTAG-$5^\prime$ overhang plus a $5^\prime$-GATC-$3^\prime$ overhang. These are complementary and base pair by hydrogen bonding. The breaks in the DNA backbones are sealed by T4 DNA ligase, which hydrolyzes ATP to energize the reaction.}\label{fig:restriction-ligation}
\end{figure}
\end{frame}

\hypertarget{dna-detection}{%
\section{DNA detection}\label{dna-detection}}

\begin{frame}{Background}
\protect\hypertarget{background}{}
\begin{itemize}
\tightlist
\item
  In 1969, Mary Lou Pardue and Joseph Gall developed a procedure by
  which they could hybridize radioactive single strands of DNA with
  complementary strands of DNA in chromosomes on glass slides.
\item
  This procedure, called in situ hybridization, Pardue and Gall were
  able to determine the chromosomal locations of repetitive DNA
  sequences.
\item
  Classical in situ hybridization involved spreading mitotic chromosomes
  on a glass slide, denaturing the DNA in the chromosomes by exposure to
  alkali (0.07 N NaOH) for a few minutes, rinsing with buffer to remove
  the alkaline solution, incubating the slide in hybridization solution
  containing radioactive copies of the nucleotide sequence of interest,
  washing off the radioactive strands that have not hybridized with
  complementary sequences in the chromosomes, exposing the slide to a
  photographic emulsion that is sensitive to low-energy radioactivity,
  developing the autoradiograph, and superimposing the autoradiograph on
  a photograph of the chromosomes.
\end{itemize}
\end{frame}

\begin{frame}{In situ hybridization}
\protect\hypertarget{in-situ-hybridization}{}
\begin{figure}
\includegraphics[width=0.65\linewidth]{./../images/insitu_hybrid_steps} \caption{Steps in in-situ hybridization}\label{fig:insitu-hybridization}
\end{figure}
\end{frame}

\begin{frame}{Fluorescent In Sity Hybridization}
\protect\hypertarget{fluorescent-in-sity-hybridization}{}
\begin{itemize}
\tightlist
\item
  A repetitive DNA sequence can be identified as satellite DNA
  (generally non-transcribing) only if the sequence has a base
  composition sufficiently different from that of main-band DNA to
  produce a distinct band during density-gradient centrifugation.
  Therefore, centrifugation cannot be used to identify all repetitive
  DNA sequences.
\item
  Today, in situ hybridization experiments are often done by using
  hybridization probes that are linked to fluorescent dyes or antibodies
  tagged with fluorescent compounds \ref{fig:fish-hybridization1}.
\end{itemize}
\end{frame}

\begin{frame}{Fluorescent In Sity Hybridization}
\protect\hypertarget{fluorescent-in-sity-hybridization-1}{}
\begin{itemize}
\tightlist
\item
  In one protocol, DNA or RNA hybridization probes are linked to the
  vitamin biotin, which is bound with high affinity by the egg protein
  avidin \ref{fig:fish-hybridization1}.
\item
  By using avidin covalently linked to a fluorescent dye, the
  chromosomal location of the hybridized probe can be detected by the
  fluorescence of the dye.
\item
  This procedure, called FISH (Fluorescent In Situ Hybridization), has
  been used to demonstrate the presence of the repetitive sequence
  TTAGGG in the telomeres of human chromosomes
  \ref{fig:fish-hybridization2}.
\item
  The FISH procedure is very sensitive and can be used to detect the
  locations of single-copy sequences in human mitotic and interphase
  chromosomes.
\end{itemize}
\end{frame}

\begin{frame}{FISH: Procedure}
\protect\hypertarget{fish-procedure}{}
\begin{columns}[T,onlytextwidth]
  \column{0.5\textwidth}

\begin{figure}
\includegraphics[width=0.8\linewidth]{./../images/fish_hybridization1} \caption{Localization of repeated DNA sequences in chromosomes by in situ hybridization performed with fluorescent probes}\label{fig:fish-hybridization1}
\end{figure}

  \column{0.5\textwidth}

\begin{figure}
\includegraphics[width=0.5\linewidth]{./../images/fish_hybridization2} \caption{The use of fluorescent dyes to localize the TTAGGG repeat sequence to the telomeres of human chromosomes and a photomicrograph demonstrating its telomeric location is shown in.}\label{fig:fish-hybridization2}
\end{figure}

\end{columns}
\end{frame}

\hypertarget{gene-cloning}{%
\section{Gene cloning}\label{gene-cloning}}

\begin{frame}{}
\protect\hypertarget{section}{}
\begin{itemize}
\tightlist
\item
  DNA technologies involve the cutting and pasting of DNA segments to
  produce new arrangements.
\end{itemize}

\begin{figure}[H]
\includegraphics[width=0.5\linewidth]{02-dna_modification_gene_cloning_rdna_technology_files/figure-beamer/rDNA-technology-1} \caption{Steps in rDNA technology}\label{fig:rDNA-technology}
\end{figure}
\end{frame}

\begin{frame}{}
\protect\hypertarget{section-1}{}
\begin{itemize}
\tightlist
\item
  The term gene cloning refers to the process of making many copies of a
  gene.
\item
  There are two general strategies used to make copies of a gene:

  \begin{itemize}
  \tightlist
  \item
    insertion of a gene into a vector that is then propagated in living
    cells
  \item
    cloning via polymerase chain reaction
  \end{itemize}
\item
  Obvious use cases of gene cloning are summarized in Table
  \ref{tab:gene-cloning-uses}.
\end{itemize}

\begin{table}

\caption{\label{tab:gene-cloning-uses}Some applications of gene cloning.}
\centering
\fontsize{6}{8}\selectfont
\begin{tabular}[t]{>{\raggedright\arraybackslash}p{8em}>{\raggedright\arraybackslash}p{42em}}
\toprule
Technique & Description\\
\midrule
DNA sequencing & Cloned genes provide enough DNA to subject the gene to DNA sequencing. The sequence of the gene can reveal the gene's promoter, regulatory sequences, and coding sequence. DNA sequencing is also important in the identification of alleles that cause cancer and inherited human disease.\\
Site-directed mutagenesis & A cloned gene can be manipulated to change its DNA sequence. Mutations within genes help to identify gene sequences, such as promoters and regulatory elements. The study of a mutant gene can also help to elucidate its normal function and how its expression may affect the roles of other genes. Mutations in the coding sequence may reveal which amino acids are important for a protein's structure and function.\\
Gene probes & Labeled DNA strands from a cloned gene can be used as probes for identifying RNA. This method of analysis, known as Northern blotting. Probes are also used to localize genes within intact chromosomes.\\
Expression of cloned genes & Cloned genes can be introduced into a different cell type of different species. This expression of cloned genes has many uses: Research, Biotechnology and Clinical trials.\\
\bottomrule
\end{tabular}
\end{table}
\end{frame}

\begin{frame}{}
\protect\hypertarget{section-2}{}
\begin{itemize}
\tightlist
\item
  A gene (or a segment of DNA containing the gene) is \textbf{removed}
  from its native site within a chromosome and inserted into a smaller
  segment of DNA known as a vector.
\item
  A vector (carrier) is a small DNA molecule that replicates
  independently of the chromosomal DNA and produces many identical
  copies of an inserted gene.

  \begin{itemize}
  \tightlist
  \item
    A vector is replicated within a living cell (called a host cell)
  \end{itemize}
\item
  Common gene-cloning vectors are plasmids or viruses.
\item
  Plasmids are found naturally in many strains of bacteria and
  occasionally in eukaryotic cells.

  \begin{itemize}
  \tightlist
  \item
    generally involved with antibiotic resistance -- R factors
  \end{itemize}
\end{itemize}
\end{frame}

\begin{frame}{Gene cloning vectors}
\protect\hypertarget{gene-cloning-vectors}{}
\bcolumns
\column{0.5\textwidth}

\begin{figure}
\includegraphics[width=0.76\linewidth]{../images/cloning_vectors} \caption{Vectors commonly used in cloning small segments of DNA.}\label{fig:cloning-vectors}
\end{figure}

\column{0.5\textwidth}

\begin{figure}
\includegraphics[width=0.98\linewidth]{../images/Ti_plasmid} \caption{The structure of Ti-plasmid.}\label{fig:ti-plasmid}
\end{figure}

\ecolumns
\end{frame}

\begin{frame}{}
\protect\hypertarget{section-3}{}
\bcolumns
\column{0.5\textwidth}

\begin{figure}
\includegraphics[width=0.86\linewidth]{../images/steps_in_gene_cloning} \caption{Steps in gene cloning.}\label{fig:steps-gene-cloning}
\end{figure}

\column{0.5\textwidth}

\scriptsize

Following ligation, the rDNA is introduced into host cells treated with
agents that render them permeable to DNA molecules. Cells that can take
up DNA from the extracellular medium are called competent cells (small
proportion of bacterial cell). This step is called transformation.

The chromosomal DNA has been inserted into a region of the vector that
contains the lacZ gene, which encodes the enzyme
\(\beta\)-galactosidase. The insertion of chromosomal DNA into the
vector disrupts the \(lacZ\) gene so it no longer produces a functional
enzyme. By comparison, a recircularized vector has a functional \(lacZ\)
gene. The functionality of \(lacZ\) is determined by providing the
growth medium with a colorless compound, X-Gal
(5-bromo-4-chloro-3-indolyl-\(\beta\)-D-galactoside), which is converted
by \(\beta\)-galactosidase into a blue dye. Bacteria grown in the
presence of X-Gal and IPTG (isopropyl-\(\beta\)-D-thiogalactopyranoside,
an inducer of the \(lacZ\) gene) form blue colonies if they have a
functional lacZ gene and white colonies if they do not. In this
experiment, therefore, bacterial colonies containing recircularized
vectors form blue colonies, whereas colonies containing recombinant
vectors are white.

\ecolumns
\end{frame}

\begin{frame}{}
\protect\hypertarget{section-4}{}
\begin{figure}

{\centering \includegraphics[width=0.56\linewidth]{../images/bac_vector} 

}

\caption{A simplified drawing of a BAC cloning vector. The vector contains several genes that function in vector replication and segregation. The origin of replication is designated $oriS$, and the $repE$ gene encodes a protein essential for replication at $oriS$. The $parA$, $parB$, and $parC$ genes encode proteins required for the proper segregation of the vector into daughter cells. A chloramphenicol resistance gene, $cm^R$, provides a way to select for cells that have taken up the vector, based on their ability to grow in the presence of the antibiotic chloramphenicol. The vector also contains unique restriction enzyme sites, such as HindIII, BamHI, and SphI, for the insertion of large fragments of DNA. These sites are located within the $lacZ$ gene, which encodes the enzyme $\beta$-galactosidase. Vectors with DNA inserts can be determined by plating cells on media containing the compound X-Gal.}\label{fig:bac-vector}
\end{figure}
\end{frame}

\hypertarget{pcr}{%
\section{PCR}\label{pcr}}

\begin{frame}{Background}
\protect\hypertarget{background-1}{}
\begin{itemize}
\tightlist
\item
  An in vitro technique that enables DNA fragments to be copied in a
  process that is referred to as \emph{amplification}.
\item
  Usefulness is because sufficient DNA can be generated to characterize
  and analyze it.

  \begin{itemize}
  \tightlist
  \item
    e.g.: tracking disease processes by characterizing genetic
    fingerprints of pathogenic organisms
  \end{itemize}
\item
  The polymerase chain reaction makes copies of dissociated DNA or
  single - stranded copies of mRNA.
\end{itemize}
\end{frame}

\begin{frame}{PCR steps}
\protect\hypertarget{pcr-steps}{}
\begin{itemize}
\tightlist
\item
  Denaturing of the target sequence so that the polynucleotide is in a
  single stranded, unwound form. For DNA, this requires a temperature of
  \(94-96^\circ\) C for 5 min.
\item
  Hybridization so that the primers bind to complementary bases that
  flank the regions on either side of the polynucleotide of interest.
  This step requires about 30 s and is carried out at \(30-65^\circ\).
\item
  Polymerization (i.e., polynucleotide or DNA synthesis) by DNA
  polymerase. The enzyme reads the template strand between the flanking
  primers, and matches the bases with complementary ribonucleotides,
  which are then incorporated into the primer strand. This is carried
  out at \(72-80^\circ\)C for several minutes at pH = 8.4 in Tris buffer
  containing ribonucleotides representing all 4 bases (A, G, C, T).
\end{itemize}
\end{frame}

\begin{frame}{}
\protect\hypertarget{section-5}{}
\begin{figure}
\includegraphics[width=0.36\linewidth]{./../images/pcr_steps} \caption{Graphical representation of amplifying a target DNA sequence through the polymerase chain reaction (PCR) [reproduced from Brown (2006), Fig. 9.2] .}\label{fig:pcr-steps}
\end{figure}
\end{frame}

\begin{frame}{PCR ingredient: Thermally tolerant DNA polymerase}
\protect\hypertarget{pcr-ingredient-thermally-tolerant-dna-polymerase}{}
\begin{itemize}
\tightlist
\item
  Polymerase from the thermophilic bacterium \emph{Thermus aquaticus}
  resolved issues of specificity and speed.
\item
  \emph{T. aquaticus} (abbreviated \emph{Taq}) is found in hotsprings
  and grows at \(75^\circ\)C. Its enzyme has an optimum temperature of
  \(72^\circ\)C and is sufficiently stable at \(96^\circ\)C so that
  fresh Taq enzyme does not need to be added after each cycle.
\item
  Taq polymerase allows primer annealing and extension to be carried out
  at temperatures significantly higher than those for E. coli
  polymerase. This reduces imperfect annealing and subsequent
  amplification of an imperfect (nonspecific) product.
\item
  This enabled researchers to design and build automated PCR machines,
  and thereby accelerated the rapid application of this technology.
\end{itemize}
\end{frame}

\begin{frame}{PCR: Classic protocol}
\protect\hypertarget{pcr-classic-protocol}{}
\begin{block}{Watson et al. ( 1992, p. 8) state:}
The standard PCR is typically done in a 50 - or 100 $\mu l$ volume and, in addition to the sample DNA, contains 50 mM KCl, 10 mM Tris/HCl (pH 8.4 at room temp.), 1.5 mM MgCl 2, 100 $\mu$g/ml gelatin, 0.25 $\mu$M of each primer, 200 $\mu$M of each deoxynucleotide triphosphate (dATP, dCTP, dGTP, and dTTP), and 2.5 units of Taq polymerase. The type of the DNA sample will be variable but it will usually have between $10^2$ to $10^5$ copies of the template (e.g., 0.1 $\mu$g DNA). A few drops of mineral oil are often added to seal the reaction and prevent condensation ... The amplification can be conveniently performed in a DNA Thermal Cycler (Perkin -Elmer Instruments) using the “Step-Cycle” program set to denature at $94^\circ$C for 20 sec, anneal at $55^\circ$C for 20 sec, and extend at  $72^\circ$C for 30 sec for a total of 30 cycles. (The “Step-Cycle” program causes the instrument to heat and cool to the target temperatures as quickly as possible. In the current instrument, this results in a heating rate of about  $0.3^\circ$C per sec and a cooling rate of about  $1^\circ$C per sec, for an overall single cycle time of approximately 3.75 min).
\end{block}
\end{frame}

\begin{frame}{PCR: Detailed methodology}
\protect\hypertarget{pcr-detailed-methodology}{}
\begin{itemize}
\tightlist
\item
  A sequence of a small stretch of nucleotides on either side of the
  target must be known, these flanking sequences are then used to
  specify and synthesize single-stranded oligonucleotide fragments, or
  primers (\textasciitilde20 bp).
\item
  A mixture of the template sequence (primers), Taq polymerase,
  triphosphorylated deoxyribonucleotides, and buffer then doubles the
  amount of DNA each cycle.
\item
  The amount of DNA is doubled during each cycle, with n cycles
  producing \(2^n\) as present to begin with.
\item
  Rate of misincorporation: 1 base out of 10,000 - 1 base out of 200,000
  per PCR cycle.
\item
  This limitation was overcome for sequencing long fragments of genomic
  DNA through use of in vivo DNA amplification using yeast or BACs.
\end{itemize}
\end{frame}

\begin{frame}{Applications}
\protect\hypertarget{applications}{}
\begin{itemize}
\tightlist
\item
  Detection of pathogenic microorganisms ( \emph{H. pylori},
  tuberculosis, chlamydia, viral meningitis, viral hepatitis, AIDs,
  etc.)
\item
  Disease epidemiology and forcasting.
\item
  Crime detection and forensic application.
\item
  Crop variety characterization by DNA fingerprinting.
\end{itemize}
\end{frame}

\hypertarget{rflp-fingerprinting}{%
\section{RFLP Fingerprinting}\label{rflp-fingerprinting}}

\begin{frame}{Steps}
\protect\hypertarget{steps}{}
\begin{itemize}
\tightlist
\item
  The DNA is cut with a restriction enzyme.
\item
  The DNA fragments are separated by length or molecular weight by gel
  electrophoresis.
\item
  The fragments are visualized by Southern blotting. The separated
  fragments are transferred from the gel to nylon paper. Then a
  radioactively labeled DNA probe is added.
\item
  The probe binds to those DNA fragments with complementary sequences.
\item
  The blot is covered with radiation-sensitive film to give an
  autoradiograph. This shows the location of those DNA fragments that
  reacted with the radioactive probe.
\item
  The final product of a DNA fingerprint is an autoradiograph that
  contains at least five essential lanes (Figure
  \ref{fig:fingerprint-lanes}). The markers are standardized DNA
  fragments of known size, which have been radioactively labeled.
\end{itemize}
\end{frame}

\begin{frame}{}
\protect\hypertarget{section-6}{}
\begin{figure}
\includegraphics[width=0.32\linewidth]{../images/rflp_fingerprinting} \caption{Outline of RFLP based DNA fingerprinting}\label{fig:rflp-fingerprinting}
\end{figure}
\end{frame}

\begin{frame}{}
\protect\hypertarget{section-7}{}
\begin{figure}
\includegraphics[width=0.4\linewidth]{../images/fingerprint_lanes} \caption{Actual DNA fingerprint showing that the pattern of DNA fragments of the victim (V) were found on the defendant's clothing (jeans/shirt). The first two and last two lanes are the standard size markers (labeled $\lambda$ and 1 kb). The lane marked TS is a positive control showing that the fingerprint technique was successful. The lane marked D is the defendant's DNA pattern.}\label{fig:fingerprint-lanes}
\end{figure}
\end{frame}

\end{document}
