---
title: "Numerical solutions to some problems in Plant Breeding"
author: |
  | Deependra Dhakal
# date: "9/18/2020"
output: 
  bookdown::pdf_document2:
    keep_tex: yes
    toc: false
    includes:
      in_header: plant_breeding_solutions_header.tex
documentclass: exam
classoption: "answers,addpoints,12pt"
geometry: "top=1.5cm,bottom=1.5cm,left=1.5cm,right=1.5cm"

---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, include = TRUE, message = FALSE, warning = FALSE)
options(knitr.kable.NA = '')
require(tidyverse)
```

# Solution to Exam Qustions


\bquestions

\question[4] \label{quest:first}

Mention different factors that affect gene and genotype frequencies. A population consisting of 10,000 individual, 25 individuals are of "aa" genotype. If the population is in Hardy-Weinberg equilibrium, find the gene and genotype frequencies of that population.

\bsolution (Question \ref{quest:first}) 

Refer to lecture note on Genetic Composition of Cross Pollinated Crops for factors affecting gene and genotype frequencies.

\underline{Numerical}

Total population ($N$) = 10000

Number of individuals of $aa$ genotype = 25

Since the population is in HW equilibrium, gene and genotype frequencies of the population stay constant over generations, i.e.

Gene frequencies = $p^2 + q^2 = 1$, and 

Genotype frequencies = $p^2AA + 2pqAa + q^2aa = 1$

So, 

$\rightarrow$ genotype frequency of homozygous recessive $aa$($q^2$) = $\frac{25}{10000}$ = `r (p = 25/10000)`

$\rightarrow$ gene frequency of minor allele $a$($q$) = $\sqrt{\frac{25}{10000}}$ = `r (q = sqrt(25/10000))`

$\rightarrow$ gene frequency of major allele $A$($p$) = $1 - p$ = `r (p = (1-q))`

$\rightarrow$ genotype frequency of heterogygous $Aa$($2pq$) = `r (2 * p * q)`

$\rightarrow$ genotype frequency of homozygous dominant $AA$($p^2$) = `r (p^2)`

\esolution

<!-- \question[4] -->
<!-- This question is worth marks 20. -->
<!-- \bparts -->
<!-- \stpart[1]  -->
<!-- What is plant breeding ? -->
<!-- \stpart[1] -->
<!-- How is plant breeding different from plant genetics ? -->
<!-- \stpart -->
<!-- This is much big a question. -->
<!-- \eparts -->

<!-- \question[20] Third new question -->

<!-- \bparts -->

<!-- \stpart This is first subpart of question -->

<!-- \bsubparts  -->

<!-- \stsubpart  -->

<!-- Is this subpart correct 3, 1, 1st subpart ? -->

<!-- \stsubpart  -->
<!-- Is this subpart correct 3, 1, 2nd subpart ? -->

<!-- \esubparts -->

<!-- \stpart This is second part of the question -->

<!-- \eparts -->

\question[4] Calculate gametic and phenotypic frequencies of GGGg individuals after selfing. Also, write types of genotype and phenotype.

\bsolution

Let us assume tetraploid from only bivalents during pairing. If we start with a GGGg plant and self it we can deduce the frequency of possible gametes for the given genotype as:


```{r diagram-gamete, fig.width=4, out.width="50%", fig.cap="Pairing of chromosomes of gametes in a tetraploid of genotype AAAa", fig.pos="H"}
DiagrammeR::mermaid(
  "
  graph TB
  
  11[A] --> 21[AA]
  12[A] --> 21
  11 --> 22[AA]
  13[A] --> 22
  11 --> 23[Aa]
  14[a] --> 23
  12 --> 24[AA]
  13 --> 24
  12 --> 25[Aa]
  14 --> 25
  13 --> 26[Aa]
  14 --> 26
  "
)
```

Here, following proportion of gametes are formed:

```{r gamete-proportion}
gametes <- combn(c("A", "A", "A", "a"), 2) %>% 
  t() %>% 
  as_tibble() %>% 
  unite(col = "gametes", sep = "") 

gametes %>% 
  count(gametes, name = "number") %>% 
  mutate(proportion_gametes = number/sum(number)) %>% 
  knitr::kable(booktabs = TRUE, caption = "Proportion of gametes formed by segregation of bivalents in a tetraploid of genotype AAAa", 
               col.names = c("Gamete", "Number", "Proportion")) %>% 
  kableExtra::kable_styling(latex_options = "HOLD_position")
```


Now, if selfing were to be performed in the individual of genotype of AAAa, following outcome of progeny genotypes is possible:

```{r}
outer(c("AA", "Aa"), c("AA", "Aa"), paste0) %>%
  as_tibble() %>%
  magrittr::set_colnames(c("AA", "Aa")) %>%
  add_column(.before = 1, female = c("AA", "Aa")) %>%
  gt::gt(rowname_col = "female") %>%
  gt::tab_spanner(label = "Male", columns = 1:3) %>%
  gt::tab_row_group(group = "Female", rows = 1:2) %>%
  gt::as_latex()

````

It could be observed that out of 3 genotypes are possible and (based on dominance hypothesis) only one phenotype results in the progeny.

\esolution

\question[4] What are advantages of partial diallel cross ? Make a partial diallel scheme involving 8 parents.
\label{quest:diallel}

\bsolution (Question \ref{quest:diallel})

Features of a diallel cross design are: 
\begin{itemize}
\item Not all the crosses are made.
\item There are no reciprocals or selfs.
\end{itemize}

The goal is to reduce the breeding workload for a given sample of parents by making less than $\frac{n(n-1)}{2}$ crosses for n parents.

With the partial scheme of diallel cross, same number of parents could be tested in a framework with fewer crosses. The number of crosses in a partial diallel scheme is given by,

$$
x = \frac{n \times s}{2}
$$

By analogy, we could have said that large number of lines could be tested with lesser crossings under diallel scheme. Rearranging the above relationship, $\large n = \frac{2x}{s}$.

Further, there is restriction as to how $s$ should is selected, for example, is $s = n -1$ it gives half diallel cross. $s$ is a whole number greater than or equal to 2, and $k$ is a whole number -- $k = \frac{n + 1 - s}{2}$.

<!-- Each parent occurs in $s$ crosses, hence the number of crosses sampled is $ns/2$. For this to be true, For k to be a whole number, we do not want both $n$ and $s$ to be odd or both even. -->

<!-- Now let's return to our previous example, with number of parents ($n$) to be 10. With $n = 10$ we definitely want each parent to occur in at least 2 sets of crosses. But, note that number of parents too is positive (10). Hence, we even values of $s$ is not allowed. Thus value of $s$ could be anywhere in $\{3, 5, 7, 9\}$. We could also verify that even values of $s$ does not satisfy the requirement for $k$ to be a wholenumber. -->

<!-- For $s = 2$, $k = \frac{n + 1 - s}{2} = \frac{10 + 1 - 2}{2} = \frac{9}{2} \neq \textrm{whole number}$ -->

<!-- For $s = 4$, $k = \frac{n + 1 - s}{2} = \frac{10 + 1 - 4}{2} = \frac{7}{2} \neq \textrm{whole number}$, and so on. -->

```{r partial-diallel-calculated}
partial_dial <- function(n, s, display_mat = c("upper", "lower", "reciprocal")) {
  n <- as.integer(n) # total number of parents
  s <- as.integer(s) # sets of crossess for parent "j" in 1:n
  k <- (n + 1L - s)/2L # possible whole number values for given "n" and "s"
  display <- match.arg(display_mat)
  
  stopifnot( all(k == floor(k)) ) # ensure k is integer
  parent_names <- paste("p", 1:n, sep = "")
  
  partial_matrix <- matrix(nrow = s, ncol = n)
  
  for(i in 1:s) {
    for(j in 1:n) {
      if(display == "reciprocal"){
        if(j > (n-k-i+1) )
          partial_matrix[i, j] <- paste(j, "x", (i+k+j-1)-n)
        else
          partial_matrix[i, j] <- paste(j, "x", i+k+j-1)
      }
      if(display == "upper"){
        if(j < (n-k-i+1) )
          partial_matrix[i, j] <- paste(j, "x", i+k+j-1)
      }
      else
        if(j > (n-k-i+1) )
          partial_matrix[i, j] <- paste(j, "x", (i+k+j-1)-n)
    }
  }
  colnames(partial_matrix) <- parent_names
  return(partial_matrix)
}

diallel_comparison_partial <- tribble(
  ~"design", ~"s",
  "partial", 2, 
  "partial", 3, 
  "partial", 4, 
  "partial", 5, 
  "partial", 6, 
  "partial", 7, 
  "partial", 8, 
  "partial", 9, 
  "full", NA, 
  "half", NA
) %>% 
  mutate(
    number_of_crosses = case_when(
    design == "partial" ~ 10 * s/2,
    design == "full" ~ 10 * 10, 
    design == "half" ~ 10 * (10-1)/2, 
    TRUE ~ NA_real_), 
    k = case_when(
      design == "partial" ~ (10 + 1 - s)/2,
      TRUE ~ NA_real_)
    )
  
# diallel_comparison_partial %>%
#   knitr::kable(booktabs = TRUE, col.names = c("Design", "$s$", "Number of Crosses", "$k$"), escape = FALSE) %>%
#   kableExtra::kable_styling() #%>%
  # kableExtra::collapse_rows(1:2, row_group_label_position = "stack", latex_hline = "major")

```

Here (Table \ref{tab:partial-diallel-8p-table}) is for example diallel cross involving 8 parental lines and 5 set of cross involving each parent,

```{r partial-diallel-8p-data}
# # # generate partial design tidily ?
# diallel_comparison_partial %>%
#   filter(design == "partial") %>%
#   filter((s %% 2) == 1)

partial_diallel_8p <- partial_dial(n = 8, s = 5, display_mat = "lower") %>% 
  as_tibble() %>% 
  pivot_longer(cols = 1:8, names_to = "parent1", values_to = "cross") %>% ## using external vector is uncommon for dplyr so add `all_of`
  dplyr::select(-parent1) %>% 
  dplyr::filter(!is.na(cross)) %>% 
  separate(col = cross, into = c("p1", "p2"), sep = c(" x "), remove = FALSE) %>%
  mutate_at(vars(p1, p2), as.numeric) %>% 
  complete(expand_grid(p1 = 1:8, p2 = 1:8)) %>%
  pivot_wider(names_from = p1, values_from = cross) %>% 
  dplyr::rename("p" = p2)

```


```{r partial-diallel-8p-table}
partial_diallel_8p %>% 
  knitr::kable(booktabs = TRUE, caption = "Partial diallel cross involving 8 parents combined in 5 set of crossess each.", escape = FALSE, longtable = TRUE) %>% 
  kableExtra::kable_styling(latex_options = c("striped", "HOLD_position"), font_size = 8)
```

\esolution

\question[4] For a quantitative trait in RMP, mean is 100 and variation is 240. The regression of the offspring on mid-parent value is 0.25. Truncation selection is practiced with a selection differential of 32. What is the expected mean in the next generation ?

\bsolution

<!-- Selection will only be successful if there is sufficient phenotypic variation and if at least some of this variation is genetic in origin. Response to selection is related to heritability. -->

<!-- $$ -->
<!-- X_1 - X_{n-1} = R = i\sigma h^2 -->
<!-- $$ -->
<!-- Where, $X_1$ is the mean phenotype of progenies obtained from breeding of selected genotypes, $X_{n-1}$ is the mean phenotype of the base population, R is the advance as a result of one round of selection, $h^2$ is the heritability (narrow sense for sexually reproducing species, or broad sense for clonally or apomictic breeding crops), $\sigma$ is the phenotypic standard deviation of the whole population, and $i$ is the intensity of selection. -->

<!-- The intensity of selection is related to the proportion (or percentage) of the population that is selected (k), and takes the values: -->

<!-- ```{r} -->
<!-- knitr::kable(tibble::tribble( -->
<!--   ~"proportion selected($k$)", ~"$i$",  -->
<!--   0.01, 2.665,  -->
<!--   0.05, 2.063,  -->
<!--   0.10, 1.755,  -->
<!--   0.20, 1.400 -->
<!-- ) -->
<!--   , caption = "This is first of mtcars table", -->
<!--              booktabs = TRUE, format = "latex", escape = FALSE) %>% -->
<!--   kableExtra::kable_styling(latex_options = c("HOLD_position")) -->
<!-- ``` -->

<!-- In cases where the initial population is large (i.e. greater than 50 genotypes) and the proportion of genotypes selected is less than 20%, then the following equation can provide an estimate of $i$. -->

<!-- $$ -->
<!-- i = 0.77 + \left(0.96 \times log\left(\frac{1}{k}\right)\right) -->
<!-- $$ -->

<!-- It can be seen that there is not a linear relationship between higher selection rates (k) and greater response to the selection applied. Retaining 10% of the selected population results in an intensity of selection value of 1.755, while retaining only 1% (i.e. a 10 fold reduction in selections) results in an intensity of selection value fo only 1.52 times larger (i.e., $i = 2.665$). -->

\underline{Numerical}

Given,

Mean ($X_0$) = `r (X0 = 100)`

Variation ($\sigma^2$) = `r (sigma_sq = 240)`

Regression of offspring on mid-parent value ($b$) = `r (b = 0.25)`

Selection differential ($X_s - X_0$) = `r (S = 32)`

We know that the performance of a selected population after applying selection differential:

\begin{align}
S &= \sqrt{\sigma^2} i \notag \\
  &= \sqrt{240} i \notag \\
i &= \frac{S}{\sqrt{240}} \notag
\end{align}

Thus, $\rightarrow$ $i$ = `r (S/sqrt(sigma_sq))`. And we know that $b = h^2_{ns}$, so:

$\rightarrow$ $R = \sigma i h^2$ = `r (R = sqrt(sigma_sq)*S/sqrt(sigma_sq)*b)`

Hence, the expected mean of next (progeny) generation ($P$) = $X_0 + \sigma i h^2$ = `r (X0 + R)`.

\esolution
\equestions
